{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4f1d2387",
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "import os\n",
    "\n",
    "sys.path.append(os.path.abspath(os.path.join(os.getcwd(), \"..\")))\n",
    "\n",
    "from app.retriever.pubmed.pubmed_client import fetch_pmc_full_paper\n",
    "from app.retriever.pubmed.pubmed_parser import extract_text_from_pmc_xml\n",
    "from app.retriever.pubmed.pubmed_retriever import fetch_pmc_papers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "7d270f26",
   "metadata": {},
   "outputs": [],
   "source": [
    "def pipeline(query: str, top_k: int = 5):\n",
    "    list_of_pmc_papers = fetch_pmc_papers(query=query, top_k=top_k)\n",
    "\n",
    "    for paper in list_of_pmc_papers:\n",
    "        pmcid = paper\n",
    "        xml_text = fetch_pmc_full_paper(pmcid=pmcid)\n",
    "        full_text = extract_text_from_pmc_xml(xml_text=xml_text)\n",
    "        print(f\"PMC ID: {pmcid}\")\n",
    "        print(full_text)\n",
    "        print(\"---------------------------------\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "36e19f28",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Searched 22 papers. Found 5 papers with PMC IDs.\n",
      "PMC ID: PMC12018185\n",
      "To the Editor: Skin pain in mild-moderate psoriasis (PsO) is under-recognized and under-investigated with existing validated questionnaires in dermatology. The Dermatology Life Quality Index (DLQI), for example, groups skin itch and skin pain together into a single question, limiting the ability to isolate the impact of pain specifically.1 Skin pain may be more prevalent in those with comorbid psoriatic arthritis (PsA),2 yet studies investigating skin pain qualities in PsO often exclude patients with pain-modifying comorbidities, such as PsA or psychiatric conditions.3 This limits insight into the real-world severity and types of pain experienced by a significant proportion of patients with PsO. We conducted a cross-sectional survey study using the Pain Quality Assessment Scale (PQAS) to better characterize the skin pain experience among patients with mild-moderate PsO and PsA. Methods are described in the Supplementary Data, available via Mendeley at https://data.mendeley.com/datasets/9x96vxp2cs/1.\n",
      "\n",
      "In total, 64.9% (87/134) of solicited patients were included in this study (44.8% male, 77% White, 59.8 ± 12.32 years (Supplementary Table I, available via Mendeley at https://data.mendeley.com/datasets/9x96vxp2cs/1)). Mean Body Surface Area was 0.51 ± 0.83 (range 0% to 5%), mean Psoriasis Area Severity Index was 0.77 ± 1.61 (range 0-8.7), and mean DLQI was 2.02 ± 3.20. 14.9% of participants had nail PsO, 6% had palmoplantar PsO, 10.7% had scalp PsO, and 10.3% had fibromyalgia. 79.3% (69/87) of patients reported skin pain and were younger (58.4 ± 11.9 vs 65.0 ± 13.0 years, P = .043) with worse DLQI scores (2.4 ± 3.5 vs 0.5 ± 1.0, P = .031) compared to those without skin pain.\n",
      "\n",
      "Pain quality patterns across PQAS questions remained generally consistent between gender, scalp, palmoplantar, and fibromyalgia subgroups, but differed in severity (Fig 1). The most common pain qualities were “aching” (69.6%), “tender” (69.6%), “dull” (65.2%), “itchy” (65.2%), and “hot” (59.4%); “cold” (26.1%) was least common (Table I). Global intensity (3.96 ± 2.97) and/or unpleasantness (4.48 ± 3.31) scores were generally higher than any individual quality, and like previous studies, were significantly higher in females vs males (intensity P = .003; unpleasantness P = .004).4 Patients with scalp and palmoplantar PsO reported higher average “itchy” (3.38 ± 2.83) and “hot” (3.50 ± 2.65) scores respectively, while those with fibromyalgia reported higher scores across most pain qualities.Fig 1Psoriatic skin pain profiles across subgroups with clinical features of interest. PQAS, Pain quality assessment scale.Table ISkin pain characteristics in patients with mild-moderate PsO and PsAN = 69 (%)Mean (SD)Pain qualities ranked by prevalence Aching, n48 (69.6)3.4 (3.02) Tender, n48 (69.6)3.10 (3.01) Dull, n45 (65.2)3.16 (3.16) Itchy, n45 (65.2)2.26 (2.53) Hot, n41 (59.4)2.30 (2.74) Sharp, n40 (58)3.00 (3.20) Radiating, n40 (58)2.45 (2.84) Throbbing, n38 (55.1)2.52 (3.07) Shooting, n38 (55.1)2.45 (2.91) Tingling, n36 (52.2)1.77 (2.20) Heavy, n34 (49.3)2.16 (2.92) Sensitive, n34 (49.3)1.87 (2.50) Numb, n33 (47.8)1.65 (2.45) Cramping, n30 (43.5)2.12 (2.98) Electrical, n29 (42)1.64 (2.54) Cold, n18 (26)0.64 (1.38)Global pain measures Intense, n60 (87)3.96 (2.97) Unpleasant, n60 (87)4.48 (3.31)Spatial descriptors Intense surface, n52 (75.4)3.04 (2.86) Intense deep, n51 (73.9)3.93 (3.30)Temporal descriptors Intermittent pain, n33 (47.8)N/A Variable pain, n31 (44.9)N/A Stable pain, n5 (7.2)N/APsO, Psoriasis; PsA, psoriatic arthritis.\n",
      "\n",
      "Global skin pain measures significantly correlated with functional outcomes (SF36_BP, SF36_PCS ρ = 0.40-0.52), but not DLQI (ρ = 0.20-0.22) or PASI (ρ = 0.12-0.14). Disease Activity index for Psoriatic Arthritis (DAPSA) also significantly correlated with global skin pain measures (ρ = 0.56-0.57, P < .05), possibly suggesting a modulatory effect of PsA on overall skin pain burden. Conversely, “itchy” pain was moderately and weakly correlated with DLQI (ρ = 0.52) and PASI (ρ = 0.36), respectively, but lacked functional correlation. Body surface area did not correlate with any PQAS question.\n",
      "\n",
      "These findings provide greater insight into the psoriatic skin pain experience. With growing evidence of central sensitization in psoriatic disease,5 integrating pain management approaches that target both peripheral and central pain mechanisms may be particularly beneficial for certain patients. This study is limited by potential sampling bias through single-institution recruitment and was underpowered to test for significance for some parameters. Given these limitations, further research is needed to evaluate the role of PQAS-specific screening in improving outcomes for patients with psoriatic disease.\n",
      "\n",
      "\n",
      "**Conflicts of interest**\n",
      "\n",
      "Dr Hanna has participated in clinical trials with Abbvie, Acelyrin, Akros, Allergan, Amgen, Apogee, Arcutis, Bausch Health, BMS, Boehringer-Ingelheim, Biopharma, Cara, Caliway, Celgene, Coherus, Concert Pharma, Cutanea, Dermira, Galapagos, Galderma, Glenmark, Incyte, Innovaderm, Janssen, Leo, Lilly, L’Oreal, Lumenis, Merz, Nektar, Novartis, Opsidio, Pfizer, Prollenium, Regeneron, Revanesse, Reistone, Sanofi, UCB; received payment from Abbvie, Allergan, Amgen, Altius Healthcare, Apogee, Aralez, Bausch Health, Caliway, Cara, Celgene, Galderma, L’Oreal, Leo, Lilly, Merz, Novartis, Paladin, Pedia-Pharm, Pfizer, Prollenium, Sandoz, Sanofi, Sun Pharma, UCB, and Vichy. Dr Chandran has received research grants from AbbVie, Amgen, and Eli Lilly; has received honoraria for advisory board member roles from AbbVie, Amgen, BMS, Eli Lilly, Janssen, Novartis, and UCB; and his spouse is an employee of AstraZeneca. Authors Dienes, Rahmani, Gao, and Dr Kashetsky have no conflicts of interest to declare.\n",
      "---------------------------------\n",
      "PMC ID: PMC12018194\n",
      "Bimekizumab, a monoclonal antibody selectively inhibiting interleukin (IL) 17A and IL-17F, was Food and Drug Administration–approved in 2023 for the treatment of moderate-to-severe plaque psoriasis in adults. Clinical trials have demonstrated a comparable safety profile to other IL-17A antagonists, with a slightly increased risk of fungal infection and the additional warning for suicidal ideation and behavior.1, 2, 3 Post-hoc analyses of adverse events (AEs) associated with bimekizumab outside of clinical trials are scarce, with no studies to date published in the United States.4,5 Our study aims to address this gap by characterizing the tolerability of bimekizumab among US adult psoriasis patients.\n",
      "\n",
      "We conducted a retrospective cohort study of adult patients with psoriasis treated with bimekizumab at 2 academic centers from November 2023 to October 2024. Specialty pharmacy records were used to identify patients. Charts were reviewed in the electronic medical record to collect data on patient demographics, AEs, and treatment response. All patients were included in the analysis of tolerability, while efficacy data were collected only for patients on bimekizumab for 3 or more months. Descriptive statistics were calculated.\n",
      "\n",
      "Fifty one and 38 patients met inclusion criteria for analysis of AEs and efficacy, respectively. Patient demographics included 33 patients (65%) with concomitant psoriatic arthritis and 4 (8%) with comorbid hidradenitis suppurativa (Table I). Overall, 71% (27/38) experienced improvement of psoriasis on bimekizumab, and 76% (19/25) had improvement in psoriatic arthritis (Table II). Key findings include 27 patients (53%) experiencing at least 1 AE related to bimekizumab. The most common AE was fungal infection (27%)—including thrush (18%), intertrigo (10%), and vulvovaginal candidiasis (6%)—with some patients developing multiple infections. Upper respiratory infection occurred in 10% of patients and COVID-19 in 6%. Other infections (8%) included pyelonephritis (4%), cellulitis (4%), and folliculitis (2%). Additional AEs included injection site reactions (10%), eczematous eruption (8%), worsening psoriasis (8%), and transaminitis (2%). One patient (2%) developed new-onset depression due to bimekizumab, but none reported suicidal ideation or behavior. Fifteen patients (29%) ultimately discontinued bimekizumab—7 (14%) for intolerable AEs, 7 (14%) for inadequate effect, and 2 (4%) for insurance reasons.Table IDemographic dataEntire cohort (n = 51)Age, mean (range)55 (22-82) ySex, n (%) Female25 (49) Male26 (51)Race, n (%) Asian3 (6) Black or African American2 (4) White43 (84) Unknown3 (6)Ethnicity, n (%) Hispanic or Latino4 (8) Not Hispanic or Latino44 (86) Unknown3 (6)Insurance type, n (%) Commercial40 (78) Medicaid1 (2) Medicare10 (20)Diagnoses, n (%) Psoriasis51 (100) Psoriatic arthritis33 (65) Hidradenitis suppurativa4 (8)Table IIBimekizumab treatment and adverse event dataEntire cohort (n = 51)No. of doses received, mean (SD)∗5.0 (1.8)Duration of treatment, mean (SD)4.7 (2.7) moImprovement in psoriasis, n (%)27 of 38 (71)Improvement in psoriatic arthritis (if applicable), n (%)19 of 25 (76)Adverse events, n (%) Any adverse event(s)27 (53) Superficial fungal infection14 (27) Thrush9 (18) Intertrigo5 (10) Vulvovaginal candidiasis3 (6) Upper respiratory infection5 (10) COVID-19 infection3 (6) Other infection4 (8) Dental infection1 (2) Urinary tract infection2 (4) Skin or soft tissue infection2 (4) Depression or worsening mood1 (2) Suicidal ideation or behavior0 (0) Injection site reaction5 (10) Pain4 (8) Swelling1 (2) Redness1 (2) Headache2 (4) Fatigue1 (2) Nausea or vomiting0 (0) Hypersensitivity0 (0) IBD development or flare0 (0) Transaminitis1 (2) Eczematous eruption4 (8) Worsening psoriasis4 (8)Discontinued bimekizumab, n (%)15 (29) Adverse events7 (14) Inadequate effect7 (14) Insurance reasons2 (4)IBD, Inflammatory bowel disease; SD, standard deviation.∗All but 1 patient received weight-appropriate psoriasis dosing of bimekizumab; 1 patient received psoriatic arthritis dosing.\n",
      "\n",
      "This study represents the largest observational post-hoc analysis of AEs associated with bimekizumab in adult psoriasis patients in the United States. Our results align with existing literature demonstrating fungal and upper respiratory infections as the most common AEs,1, 2, 3 reinforcing the value of directed counseling on these side effects when prescribing bimekizumab. The rate of thrush in our cohort was similar to clinical trials (12% to 19%), while upper respiratory infection was less common than the 24% to 39% seen in trials.1,2 The rate of serious AEs—including psychiatric symptoms, hepatic dysfunction, or development of inflammatory bowel disease—was exceedingly rare. Despite this, 14% of our cohort discontinued bimekizumab due to AEs, compared to 4% in clinical trials.1,2 This finding underscores that even milder side effects may limit patient acceptance and tolerability in real-life settings. Study limitations include variable documentation, infrequent follow-up, and recent bimekizumab initiation for some patients. Further investigations are warranted to continue evaluating real-world safety data for this novel therapeutic.\n",
      "\n",
      "\n",
      "**Conflicts of interest**\n",
      "\n",
      "Dr LaChance has received research funding from Pfizer and Merck and consulting funds from J&J, MedaCorp, Guidepoint, and TD Cowen unrelated to the current manuscript. Authors Zangenah, Vazquez-Machado, Loranger, Elshaboury, and Dr Hashemi have no conflicts of interest to declare.\n",
      "---------------------------------\n",
      "PMC ID: PMC12020883\n",
      "\n",
      "**Introduction**\n",
      "\n",
      "The CHOP (cyclophosphamide, doxorubicin, vincristine, and prednisone) chemotherapy regimen has been the cornerstone of diffuse large B cell lymphoma (DLBCL) first-line induction chemotherapy for three decades. However, even with the addition of the anti CD20 antibody rituximab, a significant fraction of DLBCL patients (30%–40%) are refractory or relapse after first-line treatment.1\n",
      "\n",
      "Anthracyclines and cyclophosphamide exert their anticancer activity by multiple mechanisms; however, a major contribution to their cytostatic and cytotoxic effects involves the inhibition of rRNA transcription and induction of murine double minute 2 (MDM2)-dependent p53 stabilization and activation.2,3 In fact, following inhibition of rRNA synthesis, ribosomal proteins (RPs), which are no longer used for building new ribosomes, are left free to bind and inactivate MDM2, thus blocking p53 protease digestion,4,5 leading to p53 stabilization. Therefore, ribosome biogenesis (RiBi) inhibition with activation of the RP/MDM2/p53 pathway is an important mechanism that contributes to the anti-lymphoma activity of currently used chemotherapeutic drugs.6,7,8 Notably, the induction of apoptotic cell death depends on the amount of p53 stabilization—lower levels of p53 result in cell-cycle arrest, whereas higher levels result in apoptotic cell death.9 Despite these notions, and although a direct relationship between the RiBi rate and efficacy of RiBi inhibitors has been demonstrated in preclinical models,10,11 the correlation between the baseline RiBi rate and DLBCL outcome following chemoimmunotherapy treatment has not been investigated.\n",
      "\n",
      "Although the occurrence of TP53 mutations could be a potential mechanism of resistance to chemotherapy-induced RP/MDM2/p53 pathway activation, the majority of DLBCL patients display wild-type (WT) TP53,12 thus indicating that alternative mechanisms of resistance could explain the failure of anthracycline-containing regimens in a significant fraction of cases.13,14,15\n",
      "\n",
      "Several studies have demonstrated that overexpression or rearrangements of MYC and BCL-2 oncogenes are adverse prognostic predictors, identifying a patient population with intrinsic resistance to standard chemoimmunotherapy.16\n",
      "\n",
      "In recent years, BCL-2 has emerged as the main driver of poor prognosis in DLBCL. While BCL-2 expression has been repeatedly reported as a poor prognostic predictor, the additive value of MYC overexpression seems to be context-dependent. Consistent with this, BCL-2 overexpression has been described as an independent adverse prognostic predictor irrespective of cell of origin (COO) subtyping and MYC expression levels.17,18,19,20,21,22\n",
      "\n",
      "Due to its role in B cell lymphomagenesis and chemoresistance, BCL-2 has been considered an attractive therapeutic target in B cell malignancies.23,24,25\n",
      "\n",
      "Despite the established role of BCL-2 as a predictor of chemoresistance in DLBCL, specific inhibition of its anti-apoptotic activity by the BH3-mimetic venetoclax showed low efficacy in the relapsed/refractory (r/r) setting.26 Moreover, venetoclax has been investigated in combination with R-CHOP chemoimmunotherapy as a first-line treatment for DLBCL, with suboptimal results.27,28\n",
      "\n",
      "In this study, we first observed that in DLBCL cell lines, BCL-2 overexpression promoted resistance to drugs inhibiting rRNA synthesis (doxorubicin, actinomycin D [Act D], and CX-5461). However, the addition of the BH3 mimetic venetoclax could not fully reverse BCL-2 mediated resistance to RiBi inhibitors. We demonstrate that enforced BCL-2 expression is associated with a decreased baseline rRNA synthesis rate. Decreased baseline rRNA synthesis attenuates p53 stabilization and activation upon RiBi inhibitor treatment, thus promoting chemoresistance. Combinations of RiBi inhibitors and venetoclax with drugs activating p53, irrespective of RiBi inhibition (such as direct MDM2 inhibitors or etoposide), overcome BCL-2 induced resistance in vitro and in vivo by boosting p53 stabilization levels. We then evaluated the ribosome biogenesis rate in histological sections of DLBCLs from two independent cohorts. For this purpose, we quantified by image analysis the distribution of nucleoli, which are the nuclear structures where rRNA is synthesized, after selective staining with silver. Well-established evidence indicates that the quantitative distribution of nucleoli within the cell nucleus, defined as the nucleolar area, is directly and strictly related to the rRNA transcription rate.29 We observed that in DLBCL, the nucleolar area value and ribosome biogenesis rate were highly variable. We demonstrated that a low nucleolar area value, indicative of a low ribosome biogenesis rate, was associated with increased BCL-2 mRNA expression levels and was correlated with poor outcomes following first-line chemoimmunotherapy.\n",
      "\n",
      "These findings uncover alternative mechanisms of BCL-2 mediated chemoresistance, beyond its known anti-apoptotic activity, providing a rationale for novel combination strategies for BCL-2 positive DLBCL.\n",
      "\n",
      "\n",
      "**Results**\n",
      "\n",
      "\n",
      "**BCL-2 overexpression promotes resistance to inhibition of ribosome biogenesis in DLBCL cell lines**\n",
      "\n",
      "To explore the therapeutic potential of RiBi inhibition in DLBCL and to identify mechanisms of resistance, we first tested the anti-proliferative activity of CHOP chemotherapy, single agent doxorubicin (Doxo) and Act D, a well-characterized RiBi inhibitor, in a panel of 12 DLBCL cell lines. We then investigated and compared the effects of these agents on rRNA synthesis rate and caspase-induced apoptosis.\n",
      "\n",
      "CHOP was modeled in vitro by combining Doxo, acrolein (a metabolite of cyclophosphamide known to inhibit rRNA synthesis),3 vincristine, and methylprednisolone. CHOP chemotherapy exerted cytotoxic effects in a dose-dependent manner (Figure S1A). As expected, TP53 mutant cell lines were less sensitive, irrespective of MYC and BCL-2 status (Figures 1A and S1B–S1D). Among TP53-WT cell lines, the BCL-2 negative cell line SUDHL-5 was the most sensitive, whereas BCL-2 positive cell lines were less susceptible to the cytotoxic effects of CHOP chemotherapy (Figures 1A and S1A–S1D). Similar results were observed with single agent Doxo and Act D (Figures 1B and S1E).Figure 1BCL-2 overexpression promotes resistance to inhibition of ribosome biogenesis in diffuse large B cell lymphoma cell lines(A) Bar graph showing antiproliferative effects of CHOP chemotherapy on 12 diffuse large B cell lymphoma (DLBCL) cell lines, measured by CellTiter-Glo assay (CTG). Cells were incubated for 24 h with vincristine 0.37 nM, doxorubicin 50 nM, acrolein 1.5 μM, and methylprednisolone 12.5 μM. Error bars represent standard deviation (SD) of triplicate experiments (n = 3). The heatmap shown below indicates the TP53, MYC and BCL-2 status and the cell of origin (COO).(B) CTG assay showing the effects of increasing doses of doxorubicin and actinomycin D on cell viability in 12 DLBCL cell lines after 24 h: in gray, TP53 mutant cell lines; in blue, BCL-2 positive/TP53 WT cell lines; in red, the BCL-2 negative/TP53 WT SUDHL-5 cell line. Error bars represent SD of triplicate experiments (n = 3).(C) qPCR analysis of 45S rRNA expression in three DLBCL cell lines incubated for 6 h with the indicated treatments: CHOP (vincristine 0.75 nM, doxorubicin 100 nM, acrolein 3 μM, and methylprednisolone 25 μM) and single agents RiBi inhibitors. Error bars represent SD of five independent experiments (n = 5). Student’s t test: ∗∗p < 0.01, ∗∗∗p < 0.005.(D) Scatterplot representing the effects of 12 h incubation with CHOP and single agent RiBi inhibitors on caspase 3/7 activation (as measured by Caspase-Glo assay) in SUDHL-5, TMD8, and SUDHL-6 cell lines. Each dot represents the mean of triplicate experiments (n = 3). Student’s t test: ∗p < 0.05.(E) Scheme of BCL-2 overexpression experiments. Western blot shows BCL-2 protein levels in SUDHL-5 cells carrying the empty vector (empty) or the BCL-2 TET-ON inducible system (BCL-2) after 96 h of incubation with 1 μg/ml doxycycline.(F) CTG assay showing the cytotoxic effects at 24 h of three doses of CHOP (vincristine 1.5-0.75-0.37 nM, doxorubicin 200-100-50 nM, acrolein 6-3-1.5 μM, and methylprednisolone 50-25-12.5 μM) in SUDHL-5 cells in the presence or absence of BCL-2. Error bars represent SD of five independent experiments (n = 5). Student’s t test: ∗∗∗p < 0.005.(G) Caspase-Glo assay showing levels of caspase 3/7 activation in SUDHL-5 cells treated for 12 h with CHOP (vincristine 0.75 nM, doxorubicin 100 nM, acrolein 3 μM, and methylprednisolone 25 μM) in the presence or absence of BCL-2. Error bars represent SD of triplicate experiments (n = 3). Student’s t test: ∗p < 0.05.(H and I) Effects of 24 h treatment with single agent RiBi inhibitors on cell viability (H) and apoptosis (I) in the presence or absence of BCL-2. Error bars represent SD of five independent experiments (n = 5). Student’s t test: ∗∗p < 0.01, ∗∗∗p < 0.005.\n",
      "\n",
      "As shown in Figure 1C, CHOP chemotherapy and single agent Doxo inhibited 45S rRNA synthesis with similar potency, indicating that Doxo was the main effector mediating the inhibition of rRNA synthesis in the CHOP regimen. Similar results were observed with low-dose Act D and CX-5461, a pleiotropic drug initially known only as an RNA polymerase I inhibitor,30 which was used as a positive control for rRNA synthesis inhibition. In contrast, the BH3-mimetic venetoclax and the chemotherapeutic drug etoposide, used as negative controls, did not exert any effect on rRNA synthesis (Figure 1C). Notably, while 45S rRNA synthesis was inhibited by RiBi inhibitors to a similar extent irrespective of TP53 and BCL-2 status, rRNA synthesis blockade translated into a significant induction of apoptosis only in the TP53 WT BCL-2 negative SUDHL-5 cell line. Similar behavior was observed with CHOP, single agent Doxo, Act D, and CX-5461, indicating a class effect (Figures 1D and S1F). Cell cycle changes induced by RiBi inhibitors were more pronounced in the BCL-2 negative SUDHL-5 cell line compared to BCL-2 positive TMD8 cells. No effects on the cell cycle were observed in TP53 mutant SUDHL-6 cells (Figure S1G).\n",
      "\n",
      "Following these observations, to investigate the role of BCL-2 in mediating resistance to RiBi inhibitors, we used a TET-on inducible system to overexpress BCL-2 (coding sequence) in the RiBi-inhibitors sensitive TP53-WT BCL-2 negative SUDHL-5 cell line. Cells were preincubated with doxycycline for 96 h and then treated with DMSO, CHOP, Doxo, Act D, or CX5461 for an additional 24 h (Figure 1E). BCL-2 overexpression significantly inhibited the cytotoxic activity of CHOP, single agent Doxo, Act D, and CX-5461, by attenuating caspase 3/7 cleavage (Figures 1F–1I). In line with these findings, BCL-2 silencing using two different short hairpin RNAs (shRNAs) sensitized BCL-2 positive cell lines (OCI-LY19 and TMD8) to the antiproliferative activity of RiBi inhibitors (Figures S2A–S2C).\n",
      "\n",
      "\n",
      "**Enforced BCL-2 expression is associated with a reduced rRNA synthesis rate in DLBCL cell lines**\n",
      "\n",
      "To investigate whether BH3-mimetics could overcome BCL-2 mediated resistance, we evaluated the effects of venetoclax in combination with Doxo, Act D, and CX-5461 in SUDHL-5 cells in the absence or presence of BCL-2. Importantly, as shown in Figure 2A, the addition of venetoclax did not fully reverse BCL-2-mediated resistance in this model. In fact, although venetoclax (at relatively high doses, 500 nM) showed some additive effects in combination with RiBi inhibitors (Doxo, Act D, or CX-5461), these combinations were significantly less effective in the presence of BCL-2 and were unable to completely abrogate BCL-2-mediated pro-survival activity.Figure 2Enforced BCL-2 expression is associated with a reduced rRNA synthesis rate in DLBCL cell lines(A) Bar graph showing the cytotoxic effects of the indicated treatments in the absence or presence of BCL-2. Cells were preincubated with doxycycline 1 μg/ml for 96 h and then treated for 24 h. Error bars represent standard deviation (SD) of five experiments. Student’s t test: ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.005. V, venetoclax; D, doxorubicin; A, actinomycin D; and C, CX-5461.(B) qPCR analysis showing the levels of BCL-2 mRNA and 45S rRNA in SUDHL-5 cells transduced with an Empty vector or a BCL-2 doxycycline (TET-ON) inducible system, treated with doxycycline for 96 h. Error bars represent SD of five independent experiments (n = 5). Student’s t test: ∗∗p < 0.01.(C) Co-immunoprecipitation (coIP) experiment showing the effects of BCL-2 overexpression on MDM2-p53-RPs binding in SUDHL-5 cells in the presence or absence of BCL-2. Cells were incubated with doxycycline for 96 h before harvesting for coIP analysis. Left: western blot analyses showing the expression levels of the indicated proteins in the initial lysates (INPUT) and the levels of MDM2-P53 or MDM2-RPs complexes immunoprecipitated with an MDM2 antibody. Right: protein quantification of the coIP experiment performed using the ImageJ software.(D) Immunoprecipitation-qPCR (IP-qPCR) analysis demonstrating the interaction between nucleolin and BCL-2 mRNA in SUDHL-5 cells. The experiment was conducted in the presence or absence of the BCL-2 coding sequence (BCL2) or the BCL2 3′ untranslated region (B-UTR). Cells were treated with doxycycline for 96 h prior to nucleolin immunoprecipitation. qPCR analysis quantified the BCL2 mRNA levels in the initial sample (INPUT) and in the nucleolin-BCL2 mRNA complexes immunoprecipitated using a nucleolin-specific antibody (IP nucleolin). Error bars represent SD of triplicate experiments (n = 3). Student’s t test: ∗∗∗p < 0.005.(E) Representative immunofluorescence images of SUDHL-5 cells showing the expression patterns of nucleolin (red) and BCL-2 (green) proteins. Cells were transduced with an empty vector (empty), BCL-2 (BCL-2), or the BCL-2 3′ untranslated region (B-UTR) using a TET-ON system and were incubated with doxycycline for 96 h.(F) Representative western blots showing total protein expression levels of nucleolin and BCL-2, under the same experimental conditions as described in Figure 2E.(G) Immunohistochemistry representative images showing three different patterns of nucleolin distribution in primary FFPE DLBCL tissues, depending on BCL-2 expression levels. Bar, 10 μm.(H) Violin plot showing the correlation between the three patterns of nucleolin distribution and BCL-2 mRNA levels, as measured by T-GEP. Student’s t test: ∗p < 0.05, ∗∗p < 0.01.\n",
      "\n",
      "To define the functional implications of these findings and to evaluate possible interactions between BCL-2 and RiBi, we first investigated the effects of enforced BCL-2 overexpression on RiBi rate. Notably, enforced BCL-2 expression resulted in decreased 45S rRNA synthesis, indicative of a reduced RiBi rate (Figure 2B). In line with this observation, BCL-2 silencing induced an increase in 45S rRNA synthesis (Figure S2D). To investigate the effects of BCL-2 overexpression on the RP-MDM2-p53 axis, we performed MDM2 co-immunoprecipitation experiments in SUDHL-5 cells in the presence or absence of BCL-2. In line with the inhibition of the 45S rRNA synthesis rate, BCL-2 overexpression resulted in increased binding of RPs RPL11 and RPL5 to MDM2 (Figure 2C), which was associated with a reduced p53-MDM2 interaction, leading to a modest upregulation of p53 protein levels that translated in a mild increase of the G1 fraction of the cell cycle in the absence of cell death, and in a slightly decreased proliferation rate (Figures S3A–S3E). Similar inhibition of 45S rRNA synthesis upon BCL-2 overexpression was observed in TP53 mutant OCI-LY7 cells, with no signs of p53 stabilization and minor effects on cell proliferation (Figures S3F–S3H). Taken together, these data indicate that BCL-2 overexpression leads to a reduced rRNA synthesis rate irrespective of the TP53 mutational status in DLBCL cell lines; however, in a TP53 WT background (SUDHL-5 cell line), these changes translate into weak activation of the RP-MDM2-P53 axis, leading to minor changes in cell proliferation and cell cycle.\n",
      "\n",
      "Following these observations, we next investigated the effects of BCL-2 overexpression on biomarkers of nucleolar stress, considering the possibility that the reduced rRNA synthesis might be due to a nucleolar stress induced by BCL-2 overexpression. Under normal conditions, the nucleolar protein nucleolin is located in the nucleolus, where it is necessary for rRNA transcription31,32; however following nucleolar stress, nucleolin translocates to the nucleoplasm. On the other hand, BCL-2 mRNA may directly interact with nucleolin through its 3′ untranslated region (UTR).33,34,35 In this context, in order to dissect the mechanisms underlying BCL-2-induced inhibition of rRNA synthesis, we used a second BCL-2 construct comprising the BCL2 coding sequence and the regulatory 3′UTR region. As shown in Figure 2D and in line with the literature, we confirmed that the full BCL-2-UTR construct directly interacted with nucleolin to a significantly higher extent compared to the BCL-2-coding sequence construct (Figures 2D and S3I). However, despite this difference, the observed phenotype was similar, as both constructs inhibited rRNA synthesis and attenuated the effects of CHOP in DLBCL cells in a similar way (Figures S3J and S3K). Immunofluorescence studies revealed that overexpression of both BCL-2 constructs induced phenotypic changes indicative of nucleolar stress in SUDHL-5 cells, as demonstrated by nucleolin delocalization from the nucleolar body to the nucleoplasm, with no effects on protein abundance (Figures 2E and 2F). These findings were confirmed by immunohistochemical analysis of 56 formalin-fixed paraffin-embedded (FFPE) DLBCL tissue samples from cohort 1 (see detailed description of patient cohorts in the STAR Methods section). By investigating the patterns of nucleolin expression with respect to BCL-2 mRNA levels measured using targeted gene expression profiling (T-GEP), we observed a significant association between BCL-2 mRNA levels and nucleolin delocalization in the nucleoplasm (Figure 2G): in samples with low BCL-2 mRNA expression, nucleolin appeared to be exclusively located in the nucleolus (pattern 1), whereas in samples with increased levels of BCL-2 mRNA, nucleolin appeared to be progressively distributed in the whole nucleus (patterns 2 and 3) (Figure 2H). Therefore, taken together, these data indicate that overexpression of BCL-2 mRNA reduces rRNA synthesis inducing a nucleolar stress that is, in turn, responsible for nucleolin translocation.36\n",
      "\n",
      "\n",
      "**BCL-2 overexpression attenuates p53 stabilization and activation following RiBi inhibitors treatment**\n",
      "\n",
      "Since the baseline level of rRNA synthesis (which is indicative of baseline ribosome biogenesis rate) correlates with the efficacy of RiBi inhibitors by regulating the amount of p53 stabilized upon drug-induced nucleolar stress,11 we next investigated the dynamic effects of RiBi inhibitor treatment on the p53 axis in the absence or presence of BCL-2, hypothesizing that a reduced baseline rRNA synthesis rate induced by BCL-2 would determine reduced levels of stabilized p53 following RiBi inhibitors exposure. As shown in Figure 3A, RiBi inhibitors (Doxo, Act D, and CX-5461) significantly downregulated rRNA synthesis irrespective of BCL-2 expression levels, in the absence of significant TP53 mRNA induction. However, in line with our hypothesis, upon drug-induced inhibition of rRNA synthesis, the amount of stabilized p53 was decreased in the presence of BCL-2 (Figure 3B). To further investigate the functional consequences of these findings, we performed RNA sequencing (RNA-seq) analyses of SUDHL-5 cells treated with RiBi inhibitors (Doxo, Act D, and CX5461) in the absence or presence of BCL-2.Figure 3BCL-2 overexpression attenuates p53 stabilization and activation following RiBi inhibitors treatment(A) qPCR analysis showing the expression levels of 45S rRNA and TP53 mRNA in SUDHL-5 cells incubated for 24 h with the indicated treatments in the presence or absence of BCL-2. Error bars represent standard deviation (SD) of triplicate experiments (n = 3). Student’s t test: ∗p < 0.05,∗∗p < 0.01.(B) Representative immunoblots showing the effect of single agents RiBi inhibitors on P53 protein levels in SUDHL-5 cells in the presence or absence of BCL-2 (same conditions as Figure 3A). Protein quantification (right) was performed using ImageJ software.(C) RNA-seq experiment performed in SUDHL-5 cells incubated with RiBi inhibitors for 6 h in the presence or absence of BCL-2 (after 96 h induction with doxycycline). The experiment was performed in triplicate (n = 3). Differentially expressed genes (DEGs) were reported in the Venn diagrams. A consensus DEGs list was extracted and matched with the TRANSFAC database to focus on differentially regulated p53 target genes and explore their expression in the absence or presence of BCL-2 following treatment with RiBi inhibitors.(D) Radar chart displaying fold changes of significantly regulated p53 target genes identified in SUDHL-5 cells treated with actinomycin D in the presence (blue line) and absence (black line) of BCL-2.(E) Bar graph showing fold change of common p53 target genes identified in SUDHL-5 cells following treatment with different RiBi inhibitors in the presence or absence of BCL-2.(F) qPCR analysis showing expression of representative p53 targets, CDKN1A (p21) and PUMA, in SUDHL-5 cells incubated 24 h with doxorubicin (100 nM), actinomycin D (2.5 nM), and CX-5461 (2,500 nM) in the presence or absence of BCL-2. Error bars represent SD of 5 independent experiments (n = 5). Student’s t test: ∗p < 0.05, ∗∗p < 0.01.(G) Representative western blots showing the effect of 24 h treatment with doxorubicin (100 nM), actinomycin D (2.5 nM), and CX-5461 (2,500 nM) on p53 protein levels in TMD8 (BCL-2 positive) and SUDHL-5 (BCL-2 negative) cells.(H) qPCR analysis showing mRNA expression levels of TP53 and CDKN1A (p21) in TMD8 and SUDHL-5 cell lines incubated for 24 h with doxorubicin (100 nM), actinomycin D (2.5 nM), or CX-5461 (2,500 nM). Error bars represent SD of 5 independent experiments (n = 5). Student’s t test: ∗p < 0.05, ∗∗p < 0.01.\n",
      "\n",
      "Principal-component analysis (PCA) revealed that BCL-2 expression was the main driver of sample segregation, followed by exposure to RiBi inhibitors (Figure S4A). Differentially expressed genes (DEGs) were identified by comparing different RiBi inhibitors (Doxo, Act D, CX-5461) versus control samples (DMSO) in the presence or absence of BCL-2 to identify candidate genes reflecting distinctive profiles. In the absence of BCL-2 (empty vector), 3,890, 3,170, and 697 DEGs were identified respectively in Doxo-, Act D-, and CX-5461-treated SUDHL-5 cells; in the presence of BCL-2, 3,641, 3,118, and 441 DEGs were identified in Doxo, Act D, and CX-5461 respectively.\n",
      "\n",
      "Detailed results of the differential expression analysis for all possible pairwise comparisons are available in the supplementary material (Data S1). Functional enrichment analysis was performed using DEGs identified in the six pairwise comparisons using clusterProfiler. According to Kyoto Encyclopedia of Gene and Genome (KEGG) pathway enrichment analysis, several pathways, including p53 signaling and cell cycle, were enriched upon treatment with RiBi inhibitors either in the absence or presence of BCL-2 (Figures S4B and S4C). However, the apoptotic pathway was enriched only in the absence of BCL-2, with 37 genes exclusively regulated in the empty vector-transduced SUDHL-5 cells (Figure S4C, and Data S2). These genes belonged to the p53 signaling, tumor necrosis factor, mitogen-activated protein kinase, and calcium signaling networks; 9 of the 37 genes were direct p53 targets. To further investigate dynamic changes in the regulation of the p53 network, which could be missed by pathway enrichment analysis, we created a DEGs consensus list and queried the TRANSFAC database (a public dataset containing a list of direct p53 targets); we obtained 256 direct p53 targets differentially expressed in the six pairwise comparisons (Figure 3C). In order to capture biologically relevant differences in the degree of activation of p53 targets in response to RiBi inhibitors treatment according to the absence or presence of BCL-2, we applied a 2-step filter: we considered only those genes with log2 fold change (fc) vs. DMSO ≥1 and with ≥0.58 log2 fc difference between absence and presence of BCL-2 in each treatment arm (Doxo, Act D, and CX-5461) (|Δfc|≥0.58log2 [Δfc = fcempty−fcBCL-2]). As shown in the representative radar chart depicted in Figure 3D, the induction of p53 targets following Act D treatment was significantly attenuated in the presence of BCL-2. Similar results were obtained with Doxo and CX-5461 (Figures 3E, S4D, and S4E).\n",
      "\n",
      "Significant attenuation of p53 targets induction (CDKN1A [p21] and PUMA) was confirmed by qPCR in SUDHL-5 cells following BCL-2 overexpression (Figure 3F).\n",
      "\n",
      "In line with these findings and with the data shown in Figure 1, the TP53 WT BCL-2 positive TMD8 cell line was characterized by attenuated p53 stabilization and target activation upon RiBi inhibitors treatment, compared to the BCL-2 negative SUDHL-5 cell line (Figures 3G, 3H, and S4F). Notably, according to our previous findings, the BCL-2 negative SUDHL-5 cell line displayed higher baseline 45S rRNA levels compared to TP53 WT BCL-2 positive DLBCL cell lines (Figure S4G).\n",
      "\n",
      "\n",
      "**MDM2 inhibitors in combination with venetoclax overcome BCL-2-mediated resistance to RiBi inhibitors in vitro and in vivo**\n",
      "\n",
      "Having demonstrated that BCL-2 attenuates p53-mediated responses to RiBi inhibitors, we investigated whether boosting p53 activation by inhibiting MDM2-p53 interactions with small molecule MDM2 inhibitors (MDM2i) would be an effective strategy to overcome BCL-2 mediated resistance.\n",
      "\n",
      "According to the underlying hypothesis, a triple combination of RiBi inhibitors (Doxo, Act D, or CX5461), nutlin 3A (MDM2i), and venetoclax was able to fully reverse BCL-2-mediated resistance in SUDHL-5 cells (Figure 4A). Interestingly, the addition of nutlin 3A to RiBi inhibitors resulted in enhanced p53 induction (which was slightly more evident after enforced BCL-2 expression) (Figures 4B and 4C), indicating a synergistic effect of MDM2i and RiBi inhibitors on p53 stabilization. In line with the restored p53 stabilization levels, a detectable caspase 3 cleavage was clearly observed with the triple combination (RiBi inhibitors + venetoclax + nutlin) in BCL-2 positive SUDHL-5 cells (Figure 4B).Figure 4MDM2 inhibitors in combination with venetoclax overcome BCL-2-mediated resistance to RiBi inhibitors in vitro and in vivo(A) CTG assay showing cell viability of SUDHL-5 cells treated for 24 h with RiBi inhibitors (doxorubicin 25, 50, 100 nM; actinomycin D 0.65, 1.25, 2.5 nM; and CX-5461 625, 1250, 2,500 nM), venetoclax (125, 250, 500 nM), and with the MDM2i nutlin-3A (1,250, 2,500, 5,000 nM) as single agents or in different combinations, in the presence or absence of BCL-2. Error bars represent standard deviation (SD) of triplicate experiments (n = 3). Student’s t test: ∗p < 0.05, ∗∗p < 0.01.(B) Representative western blot showing the effects of the indicated treatments on p53 protein abundance and caspase 3 cleavage in SUDHL-5 cells in the presence or absence of BCL-2. Cells were pre-treated with doxycycline for 96 h and then incubated with doxorubicin 100 nM, actinomycin D (2.5 nM), CX-5461 (2,500 nM), venetoclax (500 nM), nutlin-3A (2,500 nM), and the indicated combinations for 24 h.(C) Bar graph depicting p53 densitometry analysis of the immunoblot shown in Figure 4B. Densitometry analyses were performed using ImageJ software.(D) CTG assay showing cell viability of TMD8 cells treated for 24 h with RiBi inhibitors (doxorubicin 25, 50, 100 nM; actinomycin D 0.65, 1.25, 2.5 nM), venetoclax (125, 250, 500 nM), and the MDM2i nutlin-3A (1,250, 2,500, 5,000 nM) as single agents or in combination. Error bars represent SD of triplicate experiments (n = 3). Student’s t test: ∗∗p < 0.01.(E) In vivo combination experiment in a subcutaneous TP53 WT/BCL-2 positive DLBCL PDX model (LNH1). NSG mice were treated with vehicle, 0.04 mg/kg actinomycin D, 50 mg/kg venetoclax, and 100 mg/kg idasanutlin (MDM2i) as single agents or in combination. Tumor volume was measured using a caliper. Error bars represent SD of five mice (n = 5). The representative western blot shows BCL-2 baseline protein level in the LNH1 PDX, as well as in SUDHL-5 and OCILY-18 cell lines (used as negative and positive controls, respectively).(F) Bar graph showing tumor volume at day 24 (same experiment Figure 4E). Error bars represent SD of five mice (n = 5). Student’s t test: ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.005.(G) Representative image of three mice and their tumors at day 24 comparing vehicle-treated versus triple combination treated (actinomycin D, venetoclax, and idasanutlin).\n",
      "\n",
      "According to this p53-dependent mechanism of action, the triple combination of RiBi inhibitors (Doxo or Act D), nutlin 3A, and venetoclax showed enhanced efficacy, with increased p53 induction and enhanced apoptosis also in the TP53 WT TMD8 cell line (Figures 4D and S5A), but not in TP53 mutant SUDHL6 cells (Figure S5B).\n",
      "\n",
      "To further corroborate these findings, we assessed whether similar results could be obtained using different drugs inducing p53-mediated responses via RiBi-independent mechanisms, such as etoposide.37 In line with this hypothesis, the addition of etoposide to a combination of RiBi inhibitors and venetoclax reversed BCL-2 mediated resistance in SUDHL-5 cells (Figures S5C and S5D).\n",
      "\n",
      "We next investigated the efficacy of triple RiBi-BCL2-MDM2 inhibition in vivo in a subcutaneous patient-derived xenograft (PDX) model of DLBCL.38 This PDX, obtained from a nodal biopsy performed at relapse in a DLBCL patient treated with first-line R-CHOP chemoimmunotherapy, was characterized by high BCL-2 expression (Figure 4E) and a lack of genomic alterations in TP53. After PDX engraftment, NOD.Cg-PrkdcSCIDIl2rgtm1Wjl/SzJ (NSG) mice were treated with Act D, venetoclax, and idasanutlin (MDM2i) as monotherapy or in combination with different regimens (Act D + venetoclax, idasanutlin + venetoclax, Act D + idasanutlin, and Act D + idasanutlin + venetoclax) for 14 days (Figures 4E–4G). In line with our in vitro data, the triple combination of Act D, venetoclax, and idasanutlin exerted synergistic effects in this model, significantly reducing tumor growth compared to doublet combination (Act D + venetoclax, idasanutlin + venetoclax, Act D + idasanutlin) (Figures 4E–4G). We did not observe significant weight loss in mice treated with Act D, idasanutlin, and venetoclax, as single agent or in combination (Figures S5E and S5F).\n",
      "\n",
      "\n",
      "**BCL-2 overexpression is associated with decreased nucleolar area and adverse outcome in DLBCL**\n",
      "\n",
      "In order to investigate the relationship between known prognostic factors, BCL-2 mRNA levels, RiBi rate and disease outcome, we profiled an exploratory cohort of 83 DLBCL patients treated with anthracycline-based first-line chemoimmunotherapy (cohort 1, see STAR Methods), with T-GEP (for the assessment of the COO, and measurement of BCL-2 and MYC mRNA levels) and with silver staining of histological sections to determine the nucleolar area. In fact, well-established evidence indicates that the nucleolar area is directly and strictly related to the rRNA transcription.29\n",
      "\n",
      "As shown in Table 1, nucleolar area was highly variable from case to case ranging from 1.37 micron2 (μ2) to 6.71 μ2. No significant associations were observed between COO and nucleolar area (Figure S6A). Notably, we observed an inverse relationship between BCL-2 mRNA levels and nucleolar area, with higher BCL-2 mRNA levels associated with decreased nucleolar area (Figures 5A–5C). High and low gene expression levels and high and low nucleolar area values were defined based on the respective median values.Table 1Patient characteristicsExploratory cohortValidation cohortCombined cohortN° patients8346129AgeMedian, range67 (21–87)53 (23–61)60 (21–87)COO HansGCNon GC374616305376COO T-GEPGCABCUnclassified6281323158852321AgNOR area (μ2)Median, range3.38 (1.37–6.71)4.67 (1.44–9.58)3.55 (1.37–9.58)aaIPI score12385025–361088635DEXP (IHC)YesNo216213333495FISH statusMYCBCL-2BCL-6DHIT57262581751015437DEXP T-GEPYesNo216214323594BCL-2 mRNAMedian, range9.86 (5.88–12.62)8.99 (6.03–11.68)9.30 (5.88–12.62)MYC mRNAMedian, range10.22 (7.51–13.97)8.98 (7.31–10.77)9.83 (7.31–13.97)P53 (IHC)PositiveNegativeNE175336382231015Figure 5BCL-2 overexpression is associated with decreased nucleolar area and adverse outcome in diffuse large B cell lymphoma(A) Dot plot graph showing BCL-2 mRNA expression levels (measured by T-GEP) in patient samples (exploratory cohort 1, n = 83) with low and high nucleolar area, as determined by quantitative image analysis of silver-stained nucleolar structures. The definition of “high” and “low” nucleolar area was based on the median value of the nucleolar area in the whole cohort. Student’s t test: ∗∗p < 0.01.(B) Dot plot graph showing values of nucleolar area in patient samples (exploratory cohort 1, n = 83) characterized by low and high BCL-2 mRNA levels. The definition of “high” and “low” BCL-2 mRNA expression was based on the median value of BCL-2 mRNA expression in the whole cohort as determined by T-GEP. Student’s t test: ∗p < 0.05.(C) Representative nucleolar silver staining of DLBCL histological sections from two patients with high (left) and low (right) BCL-2 mRNA expression. Scale bar, 10 μm.D) Dot plot graph showing the correlation significance between mRNA levels of several BCL-2 family members and nucleolar area in the exploratory cohort. Among all BCL-2 family members, only BCL-2 mRNA levels show a significantly association with nucleolar size.(E) Dot plot graph showing nucleolar area values in patient samples (validation cohort 2, n = 46) characterized by low and high BCL-2 mRNA levels. Student’s t test: ∗∗∗p < 0.005.(F) Dot plot graph showing nucleolar area values in patient samples (cohort 1 + cohort 2, n = 129) according to BCL-2 and MYC mRNA levels. L indicates “low,” and H indicates “high.” Student’s t test: ∗p < 0.05, ∗∗∗p < 0.005.(G) Progression-free survival (PFS) curve of the whole cohort (cohort 1 + cohort 2, n = 129) according to BCL-2 mRNA expression levels. p value was calculated using the log-rank test.(H) PFS curve of the whole cohort (cohort 1 + cohort 2, n= 129) according to MYC mRNA expression levels. p value was calculated using the log-rank test.\n",
      "\n",
      "To better investigate the specificity of this observation, we profiled our exploratory cohort with a custom T-GEP panel including all BCL-2 family members (Data S3); of note, BCL-2 was the only member of the BCL-2 family whose mRNA levels were significantly associated with the nucleolar area values (Figure 5D). Interestingly, among all antiapoptotic BCL-2 family members, only BCL-2 emerged as a significant predictor of poor outcome, with higher BCL-2 mRNA levels being correlated with shorter progression-free survival (PFS) following standard chemoimmunotherapy (Figure S6B).\n",
      "\n",
      "To confirm these findings, we profiled a validation cohort of 46 patients (cohort 2) from the DLCL04 study39 using T-GEP and silver staining of nucleolar structures. Again, we observed an inverse relationship between BCL-2 mRNA levels and nucleolar area (Figures 5E and S6C). Patient characteristics of discovery and validation cohorts are shown in Table 1.\n",
      "\n",
      "Next, to further dissect the relationship between MYC, BCL-2, COO, and nucleolar area values, we analyzed a combined cohort of 129 patients (cohort 1 + cohort 2). Again, no correlation was observed between nucleolar area values and COO subtyping, with no significant differences between the activated B cell (ABC) and germinal center (GC) B cell/unclassified subgroups (Figure S6D). In line with our prior findings, we observed a strong inverse correlation between BCL-2 mRNA levels and nucleolar area in this combined cohort of patients (Figure S6D).\n",
      "\n",
      "Since MYC is known as a positive regulator of ribosome biogenesis,40 we evaluated the relative influence of MYC and BCL-2 mRNA levels on nucleolar area. Importantly, although there was a positive correlation between MYC mRNA levels and nucleolar size, which did not reach statistical significance, BCL-2 mRNA levels were negatively correlated to nucleolar area values irrespective of MYC levels: in fact, while increased BCL-2 mRNA levels were associated with decreased nucleolar size in both MYC-low and MYC-high subgroups, on the contrary, MYC mRNA levels did not correlate with nucleolar area when categorizing based on BCL-2 mRNA expression levels (Figures 5F and S6E).\n",
      "\n",
      "Notably, as opposed to BCL-2, which was associated with outcome in both cohorts (Figures S7A and S7B), MYC mRNA levels were not associated with the outcome in either dataset when analyzed separately (Figures S7C and S7D), and as a whole (n = 129 patients) (Figures 5G and 5H).\n",
      "\n",
      "These data were confirmed in silico using two independent publicly available datasets from Lenz et al.41 and Sha et al.,42 where BCL-2 mRNA levels outperformed MYC as an outcome predictor in patients treated with R-CHOP chemoimmunotherapy (Figures S8A–S8D). To further elucidate the role of MYC in regulating the sensitivity to RiBi inhibition, we used the P-4936 cell line, where MYC expression is under the control of a tetracycline regulated repressible (Tet-OFF) promoter.43 We treated P-4936 cells with CHOP or single agent RiBi inhibitors (doxorubicin, Act D, and CX-5461) for 24 h in the presence or absence of MYC. As shown in Figures S8E and S8F, in line with survival analyses depicted in Figure 5, modulation of MYC expression levels had minor effects on CHOP and RiBi inhibitors antiproliferative activity.\n",
      "\n",
      "\n",
      "**A reduced nucleolar area is an independent predictor of adverse outcome in DLBCL patients treated with standard anthracycline-based chemoimmunotherapy**\n",
      "\n",
      "Next, given the demonstrated relationship between ribosome biogenesis rate and efficacy of RiBi inhibitors in preclinical models11 and considering that inhibition of ribosome biogenesis is a relevant mechanism of action of anthracyclines, we analyzed the correlation between nucleolar area (indicative of ribosome biogenesis rate) and outcome after first-line anthracycline-based therapy in the whole cohort of 129 patients. Nucleolar area was defined as high or low based on the median value. We found that a decreased nucleolar area, indicative of a decreased ribosome biogenesis rate, was associated with poor outcome in terms of PFS and overall survival (OS) (Figures 6A and 6B). Importantly, in multivariate analyses, nucleolar area emerged as the only independent outcome predictor, irrespective of MYC, BCL-2 mRNA levels and COO subtyping (Figures 6C and 6D). Patient’s characteristics were well balanced in the nucleolar area high and low subgroups (Table S1). Finally, these data were confirmed after excluding patients with positive p53 immunostaining (n = 23, 17%), which is indicative of the presence of TP53 mutations44 (Figures S9A–S9D).Figure 6A reduced nucleolar area is an independent predictor of adverse outcome in DLBCL patients treated with standard anthracycline-based chemoimmunotherapy(A) Progression-free survival (PFS) curve of the whole cohort (exploratory + validation, n = 129) according to nucleolar size (nucleolar area). The definition of “high” and “low” nucleolar area was based on the median value of the whole cohort. p value was calculated using the log-rank test.(B) Overall survival (OS) curve of the whole cohort (exploratory + validation, n = 129) according to nucleolar size (nucleolar area). The definition of “high” and “low” nucleolar area was based on the median value of the whole cohort. p value was calculated with the log-rank test.(C) Forest plot depicting multivariable analyses for PFS in the whole cohort (exploratory + validation, n = 129).(D) Forest plot depicting multivariable analyses for OS in the whole cohort (exploratory + validation, n = 129).\n",
      "\n",
      "\n",
      "**Discussion**\n",
      "\n",
      "The R-CHOP chemoimmunotherapy regimen has been the mainstay of DLBCL treatment for decades, and even in the era of novel agents, therapeutic strategies based on small molecule inhibitors, bispecific T cell engagers and immunoconjugates, are under clinical development in combination with R-CHOP as first line therapy. Therefore, a deeper understanding of the mechanisms underlying resistance to R-CHOP chemoimmunotherapy is still timely and relevant.\n",
      "\n",
      "In the present study we demonstrate that the poor prognosis of DLBCL patients overexpressing BCL-2 is due only in part to the direct antiapoptotic activity mediated by BCL-2, but it is in fact also the consequence of reduced p53 activation upon exposure to chemotherapeutic agents inhibiting ribosome biogenesis, such as anthracyclines. We demonstrate that this reduced sensitivity is due to a downregulation of baseline rRNA synthesis rate associated with BCL-2 overexpression.\n",
      "\n",
      "Given that BCL-2 is emerging as one of the main prognostic predictors in DLBCL treated with anthracycline-containing regimens, we first investigated the impact of BCL-2 overexpression on the activity of chemotherapeutic drugs inhibiting ribosome biogenesis.\n",
      "\n",
      "We showed that CHOP treatment induced a potent inhibition of rRNA synthesis in DLBCL cell lines, this activity being mainly attributable to doxorubicin, which mirrored the effects of well-known RiBi inhibitors (such as Act D and CX-5461) on rRNA transcription. Although both doxorubicin and CX-5461 display pleiotropic mechanisms of action, inhibition of rRNA synthesis represents a well-described component of their anticancer activity, and these agents are considered potent RiBi inhibitors. According with the p53-dependent mechanism of action of RiBi inhibitors, pharmacologic inhibition of rRNA synthesis did not translate in significant cytotoxicity in TP53 mutant cell lines. However, among TP53 WT cell lines, maximal efficacy was observed in the BCL-2 negative SUDHL-5 cell line, with BCL-2 positive TP53 WT cell lines being less sensitive. Enforced BCL-2 overexpression rendered SUDHL-5 cells resistant to CHOP and single agent RiBi inhibitors treatment. This resistance was not solely due to the direct antiapoptotic activity of BCL-2, as combined treatment with the BH3-mimetic venetoclax, could not fully reverse BCL-2 induced resistance. Following these observations, we hypothesized that BCL-2 could modulate the activity of RiBi inhibitors with BH3-independent mechanisms.\n",
      "\n",
      "Indeed, following enforced BCL-2 expression we observed a reduced rRNA synthesis rate in DLBCL cells, which resulted in a modest increase in the fraction of cells in the G1 phase of the cell cycle, in the absence of apoptosis. This is in line with several early studies investigating the antiproliferative effects related to BCL-2 overexpression, showing a delayed G1-S transition in BCL-2 positive cells.45\n",
      "\n",
      "BCL-2 overexpression was associated to nucleolar changes indicative of nucleolar stress, such as the translocation of nucleolin from the nucleolus to the nucleoplasm. Considering the known direct interaction between nucleolin and BCL-2 mRNA 3′UTR,33 we hypothesized that overexpression of BCL2 mRNA might be responsible for the sequestration of nucleolin in the nucleoplasm, thus determining a reduced availability for the process of ribosome biogenesis. However, similar phenotypes were observed using a BCL-2 construct encoding only for the coding sequence and a full-length BCL-2 construct with UTRs. In fact, although we were able to confirm a direct interaction of the BCL-2-UTR construct with nucleolin, enforced expression of both constructs exerted similar effects in terms of rRNA synthesis inhibition and attenuation of CHOP-induced cytotoxicity. Immunofluorescence experiments carried out in SUDHL-5 cells were recapitulated by immunohistochemistry studies performed on FFPE tissue, confirming that, in the presence of BCL-2, nucleolin translocated from the nucleolus, to the nucleoplasm. These data suggest that BCL-2 overexpression induces a nucleolar stress that is associated, through direct or indirect mechanisms, with a reduced ribosome biogenesis.36 In this light, previous studies demonstrated the poor prognostic impact of nucleolin overexpression levels in DLBCL.46 Our data indicate that the pattern of nucleolin expression could be a useful biomarker of BCL-2-induced nucleolar stress and chemoresistance in DLBCL.\n",
      "\n",
      "In line with the known direct relationship between baseline ribosome biogenesis rate and the amount of stabilized p53 upon rRNA synthesis blockade,11 BCL-2-associated reduction of baseline rRNA synthesis resulted in attenuated p53 induction after treatment with RiBi inhibitors. RNA-seq experiments performed in SUDHL-5 cells treated with RiBi inhibitors in the presence or absence of BCL-2, indicated that enforced BCL-2 overexpression attenuated the induction of p53 target genes. Taken together these findings indicate that BCL-2 overexpression may induce chemoresistant phenotypes not only by directly inhibiting apoptosis downstream of p53, but also by attenuating p53 activation in response to RiBi inhibitors. Indeed, combinatory strategies aimed at boosting p53 activation, proved to be effective in vitro and in vivo in BCL-2 positive DLBCL models. More precisely, combinations of RiBi inhibitors with drugs stabilizing p53 in a RiBi-independent manner, such as MDM2i and etoposide, restored efficient stabilization of p53 in BCL-2 overexpressing DLBCL cells. Interestingly the combination of RiBi inhibitors and MDM2i significantly increased p53 levels as compared to either drug used as single agent. This could be explained by the fact that nutlin 3A and RPs (RPL5 and RPL11) bind MDM2 at different sites, thus determining additive effects on p53 induction.47\n",
      "\n",
      "Several prior studies have confirmed the therapeutic synergy and synthetic lethality of MDM2 and BCL-2 inhibitors through p53-dependent mechanisms.48,49 However, the present observations provide a preclinical rationale for combining RiBi inhibitors (including anthracyclines) with MDM2i to maximize p53 induction, which in turn could boost BCL-2 inhibitor-mediated apoptotic responses. Combination treatment with chemotherapeutic agents such as etoposide, that induce activation of the p53 axis irrespective of RiBi inhibition,37 had similar effects. In line with this, current evidence suggests an improved outcome with etoposide-containing regimens in double-hit high-grade B cell lymphoma or double-expresser DLBCL.50,51 Results of recent clinical trials in which anthracycline-based chemoimmunotherapy was combined with BCL-2 inhibitors further corroborate our findings: in fact, combinatory strategies based on the addition of venetoclax to R-CHOP produced suboptimal results in DLBCL, failing to provide a significant clinical benefit, as increased toxicities were not counterbalanced by increased effectiveness.27,28 These findings are consistent with a model in which restored p53-mediated responses by RiBi inhibitors-MDM2i or RiBi inhibitors-etoposide combinations could play a crucial role for the induction of synthetic lethality with BH3-mimetics in BCL-2 positive DLBCL.\n",
      "\n",
      "To determine the clinical implications of our findings, we next investigated whether differences in baseline RiBi rate may influence clinical outcomes in DLBCL patients treated with R-CHOP chemoimmunotherapy. To the best of our knowledge, the relationship between baseline RiBi rate, established biomarkers of DLBCL outcome and the efficacy of CHOP chemotherapy has never been investigated. In line with the observed effects of BCL-2 overexpression on rRNA synthesis rate, we confirmed an inverse correlation between BCL-2 mRNA levels and nucleolar area in two independent cohorts of DLBCL patients treated with anthracycline-based chemoimmunotherapy. The observed negative correlation was specific, since among all BCL-2 family members, only BCL-2 correlated with nucleolar area. Furthermore, nucleolar area was an independent prognostic predictor in multivariate analysis, outperforming BCL-2 mRNA levels. Taken together, these data demonstrate a previously unrecognized link between BCL-2 and ribosome biogenesis, suggesting that the adverse prognostic influence related to BCL-2 overexpression could be due, in relevant part, to a reduced ribosome biogenesis rate associated with BCL-2 expression, which in turn impairs p53-mediated response to chemotherapy. In this light, nucleolar area should be considered as an additional prognostic factor in DLBCL, with potential clinical relevance.\n",
      "\n",
      "Modulation of MYC levels using a TET-OFF system did not significantly affect sensitivity to CHOP or RiBi inhibitors in vitro, and in line with this MYC mRNA levels were not associated with outcome in survival analyses. These data provide a possible mechanistic explanation to the growing body of evidence suggesting a dominant prognostic role of BCL-2 overexpression and a context-dependent role of MYC in DLBCL treated with R-CHOP.18,20,21 On the other hand, it is well known that MYC is a master regulator of cancer cell proliferation, metabolism and immune evasion, with pleiotropic effects on cell cycle, DNA repair, and multiple metabolic pathways.52,53,54 It is possible that the aforementioned functions of MYC could synergize with the effects of a “second hit” (BCL-2 overexpression and/or TP53 mutation) affecting sensitivity to RiBi inhibition, thus contributing to determine the clinical aggressiveness and chemoresistant phenotypes observed in MYC/BCL-2 double-expresser lymphoma.\n",
      "\n",
      "In summary, this study uncovered alternative mechanisms of chemoresistance related to BCL-2 overexpression beyond its known anti-apoptotic function, consisting in attenuated p53-mediated responses upon blockade of rRNA synthesis induced by chemotherapy. These findings provide the rationale for novel, specific combination strategies aimed at overcoming BCL-2 induced chemoresistance in DLBCL.\n",
      "\n",
      "\n",
      "**Limitations of the study**\n",
      "\n",
      "While our findings are robust and supported by well-characterized cellular models with overexpression and knockdown approaches, PDX mouse model and comprehensive patient analyses, it is important to acknowledge possible inherent limitations, which are common in this type of study. The sample size, while sufficient for preliminary insights, may not fully represent the broader large B cell lymphoma population, thus potentially affecting the generalizability of the results. Future studies investigating the prognostic impact of nucleolar size in larger patient cohorts will be crucial to further substantiate our findings. Additionally, the inherent constraints of in vitro and in vivo models may not fully capture the complexity of lymphoma biology in the clinical setting. Finally, the precise mechanism by which BCL-2 overexpression regulates rRNA synthesis requires further elucidation.\n",
      "\n",
      "\n",
      "**Resource availability**\n",
      "\n",
      "\n",
      "**Lead contact**\n",
      "\n",
      "Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Prof. Enrico Derenzini, (enrico.derenzini@ieo.it).\n",
      "\n",
      "\n",
      "**Materials availability**\n",
      "\n",
      "This study did not generate any new unique reagents and components. The shRNA oligonucleotide sequences and primer sequences for the target genes are provided in the STAR Methods section.\n",
      "\n",
      "\n",
      "**Data and code availability**\n",
      "\n",
      "•The RNA-seq data used in this study are available at Array Express and are publicly available as of the date of publication. Publicly available datasets used in this study can be found in the Gene Expression Omnibus (GEO) database. Accession numbers are listed in the key resources table. Normalized T-GEP with associated outcome information are available upon request from the corresponding author.•The code used in this study has been deposited on Github (https://github.com/veramazzara/BCL2_rRNA_DLBCL/tree/main).•Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.\n",
      "\n",
      "\n",
      "**Acknowledgments**\n",
      "\n",
      "We thank the Genomic Unit, Computing Data, and Digital Research Platforms Unit, Imaging Unit, and Flow Cytometry Unit at IEO for their technical assistance. In particular, Simona Rodighero and Simona Ronzoni provided useful discussions and experimental support.\n",
      "\n",
      "Funding: Banca del Piemonte, Fondazione Venesio, Piaggio research grant to C.T. and E.D., AIRC 5×1000 grant no. 21198 to S.P., and AIRC IG 20109 and 10.13039/501100003196Italian Ministry of Health to F.B.\n",
      "\n",
      "\n",
      "**Author contributions**\n",
      "\n",
      "A.R. designed research studies, conducted experiments, acquired and analyzed data, and wrote the manuscript. S.M. conceptualized, designed and performed bioinformatic analyses, and critically revised the manuscript. D.S., S.Z., and E.B. conducted experiments and acquired the data. M.R.S. performed T-GEP. S.O., G.T., and P.F. conducted in vivo experiments. A.D. analyzed T-GEP data. C.Ceccarelli and C.A. performed immunohistochemistry. G.M. and F.M. performed T-GEP. V.T. performed immunohistochemistry. D.T. and M.P. performed nucleolar characterization with silver staining and analyzed the data. C. Corsini helped with in vivo experiments. A.C. helped with T-GEP experiments and analyzed the data. R.C., F.B., U.V., P.L.Z., C.T., and S.P. helped with data interpretation and critically revised the manuscript. E.D. conceptualized the research, designed research studies, analyzed data, and wrote the manuscript.\n",
      "\n",
      "\n",
      "**Declaration of interests**\n",
      "\n",
      "E.D., research funding: Takeda, ADC-Therapeutics, and Incyte; speaker’s bureau: Roche, Incyte, and Abbvie; advisory board: Astra Zeneca, Lilly, Abbvie, Roche, Gilead, Takeda, and Sobi. F.B., research support: Roche and Menarini; speaker’s bureau: Pfizer. P.L.Z., consultant: MSD, Eusapharma, and Novartis; speaker’s bureau: Celltrion, Gilead, Jassen-Cilag, BMS, Servier, MSD, Astrazeneca, Tekada, Roche, Eusapharma, Kyowa Kirin, Novartis, Incyte, and BeiGene; advisory board: Secura Bio, Celltrion, Gilead, Jassen-Cilag, BMS, Servier, Sandoz, MSD, Astrazeneca, Tekada, Roche, Eusapharma, Kyowa Kirin, Novartis, ADC Terap., Incyte, and BeiGene. S.P., speaker’s bureau: Lilly, Takeda, BeiGene, Stemline, and Roche; advisory board: Lilly, Stemline, and Diatech.\n",
      "\n",
      "\n",
      "**STAR★Methods**\n",
      "\n",
      "\n",
      "**Key resources table**\n",
      "\n",
      "REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesCleaved Caspase 3Cell Signaling TechnologyCat #9664; RRID: AB_2070042BCL-2Cell Signaling TechnologyCat#4223; RRID: AB_1903909c-MYCAbCamCat#32072; RRID: AB_731658MDM2 2A10AbCamCat#ab16895; RRID: AB_2143534VeriBlot for IP Detection Reagent (HRP)AbCamCat#ab131366; RRID: AB_2892718Beta-ActinSIGMACat#A5316; RRID: AB_476743VinculinSIGMACat#V9131; RRID: AB_477629P53 DO-1Santa Cruz BiotechnologyCat#sc126; RRID: AB_628082RPL11Bio-Rad#VMA00652RPL-5Bethyl#A303933ANucleolin 4E2GeneTexCat#GTX13541; RRID: AB_372550anti-mouse HRP-conjugatedBioradCat#170-6515; RRID: AB_11125142anti-rabbit HRP-conjugatedBioradCat#170-6516; RRID: AB_11125547BCL-2 E17AbCam#32124MDM2 SMP14AbCam#263453BCL-2 clone 124Dako AgilentCat#M0887; RRID: AB_2064429cMYC clone EP121Epitomics#MON3393Nucleolin clone 4E2AbCamCat#ab13541 RRID: AB_561053Biological samplesCohort 1: 83 patients treated with R-CHOP/CHOP-like regimensReal-life exploratory cohort, S. Orsola-Malpighi Hospital, Bologna (Italy), from 2007 to 2012Study n 12/2009/U/Tess, protocol 148/2009Cohort 2: 46 DLBCL patientsDLCL04 study (ref. 39)ClinicalTrials.gov, number NCT00499018https://clinicaltrials.gov/expert-search?term=NCT00499018Patient-derived xenografts (PDX) LNH1Laboratory of Hematology-Oncology, IEO European Institute of Oncology IRCCS, Milan, ItalyN/AChemicals, peptides, and recombinant proteinsDOXORUBICINSelleckChem#S1208ACTINOMYCIN-DBiovision#1036-50CX-5461SelleckChem#S2684NUTLIN-3Sigma-Aldrich#N6287IDASANUTLINSelleckChem#S7205ETOPOSIDESigma-Aldrich#E1383VENETOCLAX (ABT-199)SelleckChem#GDC-0199VINCRISTINEMerck Life Science#V88796a-methylprednisolone 21-hemisuccinate SMerck Life Science#M3781ACROLEINSigma-Aldrich#110221Critical commercial assaysCell Titer Glo AssayPromega#G7573Caspase-Glo 3/7 AssayPromega#G8093Quick-RNA MiniPrep kitZYMO RESEARCH#R1055LunaScript RT SuperMix kitBiolabs#E3010Deposited dataRaw RNA-seq datathis paperArrayExpress: E-MTAB-14977Expression profiling by arrayGEOGEO: GSE10846; GSE117556Experimental models: Cell linesSUDHL-4, SUDHL-6, OCI-LY-7, SUDHL10, OCI-LY-19, OCI-LY-18, OCI-LY-1DSMZACC 495, ACC 572, ACC 688, ACC 576, ACC 528, ACC 699, ACC 722SUDHL-5, SUDHL2, PfeifferATCCCRL-2958, CRL-2956, CRL-2632TMD8, HBL-1Dr. A. Younes, Memorial Sloan Kettering Cancer Center, NYN/AExperimental models: Organisms/strainsMice: NOD.Cg-PrkdcSCIDIl2rgtm1Wjl/SzJ (NSG)Charles River Laboratories#005557OligonucleotidesBCL2sh1:Fw:ACCGGTGGATGACTGAGTACCTGAACGTTAATATTCATAGCGTTCAGGTACTCAGTCATCCATTTTRv:CGAAAAAATGGATGACTGAGTACCTGAACGCTATGAATATTAACGTTCAGGTACTCAGTCATCCACThis paperN/ABCL2sh2:Fw:ACCGGGTGATGAAGTACATCCATTATGTTAATATTCATAGCATAATGGATGTACTTCATCACTTTTRv:CGAAAAAAGTGATGAAGTACATCCATTATGCTATGAATATTAACATAATGGATGTACTTCATCACCThis paperN/AMYC = Fw TTCGGGTAGTGGAAAACCAG; Rv CAGCAGCTCGAATTTCTTCCThis paperN/AACTIN = Fw GAACGGTGGTGTGTCGTTC; Rv GCGTCTCGTCTCGTCTCACTThis paperN/ABCL-2 = Fw GAGTTCGGTGGGGTCATGT; Rv GCCGGTTCAGGTACTCAGTCThis paperN/ArRNA45S = Fw GCGTCTCGTCTCGTCTCACT; Rv GAACGGTGGTGTGTCGTTCThis paperN/ATP53 = Fw GAGCTGAATGAGGCCTTGGA; Rv CTGAGTCAGGCC CTTCTGTCTTThis paperN/ACDKN1A = Fw AAATCGTCCAGCGACCTTCCTCAT; Rv TCTGACTCCTTGTTCCGCTGCTAAThis paperN/APUMA = Fw TTACTTCCTGCCCTGCTCTGGTTT; Rv TCTCAGGAGGTGCACGTTTCATCAThis paperN/ATBP = Fw CACGAACCACGGCACTGATT; Rv TTTTCTTGCTGCCAGTCTGGACThis paperN/ARecombinant DNAInDOXible Tet-Activated cDNA Lentiviral Expression SystemCellectaCUSTOMshRNA Lentiviral Vector with Tet-Inducible U6 PromoterCellecta#SVSHU6T16-LSoftware and algorithmsFastQC version 0.11.9https://www.bioinformatics.babraham.ac.uk/projects/fastqc/BBDUk- BBMap version 38.79https://archive.jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/STAR version 2.7.3ahttps://github.com/alexdobin/STARR version 4.1.1https://www.r-project.org/Githubhttps://github.com/veramazzara/BCL2_rRNA_DLBCL/tree/mainImageJ(Fiji) version 1.52ImageJhttps://fiji.sc/BD FacsDIVA v8.0.1.1BDBioscienceshttps://www.bdbiosciences.com/en-us/products/software/instrument-software/bd-facsdiva-softwareLasX version 3.5.5Leica Mycrosystems GmbHhttps://www.leica-microsystems.com/products/microscope-software/p/leica-las-x-ls/GraphPad Prism version 10.4.0GraphPad Software, CA, USAhttps://www.graphpad.com/featuresFlowJo version 9.3FlowJo, LLChttps://www.flowjo.com/\n",
      "\n",
      "\n",
      "**Experimental model and study participant details**\n",
      "\n",
      "\n",
      "**Cell lines**\n",
      "\n",
      "The human DLBCL-derived cell lines SUDHL-4, SUDHL-6, OCI-LY-7, SUDHL10 and OCI-LY-19, OCI-LY-18, OCI-LY-1 were obtained from the DSMZ-German Collection of Microorganisms and Cell Cultures, Department of Human and Animal Cell Cultures (Braunschweig, Germany); SUDHL-5, SUDHL2, Pfeiffer were obtained from ATCC (American Type Culture Collection). The DLBCL-derived cell lines, TMD8 and HBL-1, were provided by Dr. A. Younes (Memorial Sloan Kettering Cancer Center, NY).\n",
      "\n",
      "Cell lines were cultured in RPMI 1640 medium or in Iscove Modified Dulbecco’s medium supplemented with 10–20% heat-inactivated fetal bovine serum, 1% L-glutamine, and penicillin-streptomycin in a humid environment of 5% CO2 at 37°C.\n",
      "\n",
      "Cells were checked for mycoplasma contamination by PCR.55 Cell lines were authenticated by STR DNA typing (StemElite ID system) using gene amp PCR System 9700 thermal cycler (Applied biosystem) for STR amplification.\n",
      "\n",
      "\n",
      "**Mice**\n",
      "\n",
      "Experiments involving animals were approved by the Italian Ministry of Health and were performed in accordance with Italian Law (D.lgs. 26/2014 and amendments), Institutional Animal Care and Use Committee, institutional guidelines of the European Institute of Oncology, and ARRIVE guidelines. Mice were bred and housed under pathogen-free conditions in an animal facility at the IEO–IFOM Campus (Milan, Italy). In vivo studies were performed using 10-week-old immunodeficient NOD.Cg-PrkdcSCIDIl2rgtm1Wjl/SzJ (NSG) mice (Charles River Laboratories, Italy), with an equal distribution of male and female subjects.\n",
      "\n",
      "\n",
      "**Patients and study approval**\n",
      "\n",
      "In the present study, we analyzed two independent patient cohorts.\n",
      "\n",
      "\n",
      "**Cohort 1**\n",
      "\n",
      "83 patients from a real-life exploratory cohort treated with R-CHOP/CHOP-like regimens at S. Orsola-Malpighi Hospital, Bologna, from 2007 to 2012 (Study n 12/2009/U/Tess, protocol 148/2009). The patients enrolled in this study were selected based on the availability of residual tissue for AgNOR staining. However, only 56 patients from cohort 1 had tissue available for nucleolin immunohistochemistry.\n",
      "\n",
      "\n",
      "**Cohort 2**\n",
      "\n",
      "46 patients from the DLCL04 study,39 a prospective randomized phase 3 clinical trial investigating the role of first-line autologous stem cell transplant consolidation after chemoimmunotherapy in CD20+ DLBCL (ClinicalTrials.gov, number NCT00499018). All patients had available tissue for immunohistochemistry, FISH, T-GEP, and silver staining of the Nucleolar organizer regions (NOR).\n",
      "\n",
      "Patient characteristics are summarized in Table 1. This study was approved by the Institutional Review Boards of the participating centers in accordance with the Declaration of Helsinki.\n",
      "\n",
      "\n",
      "**Method details**\n",
      "\n",
      "\n",
      "**BCL2 inducible system**\n",
      "\n",
      "Inducible overexpression of the coding sequence of BCL2 in the SUDHL5 (BCL2 negative) DLBCL cell line was performed using the Cellecta InDOXible Tet-Activated cDNA Lentiviral Expression System (custom Cellecta). Two lentiviral vectors, the inducible cDNA, which is under the control of the responsive promoter (TRE) and the transactivator (rtTA), were transduced separately into SUDHL-5 and OCI-LY-7 cells, allowing gene overexpression after tetracycline analog doxycycline induction. The transactivator vector contained a puromycin resistance gene (Puro), while the TRE-Empty, the TRE-BCL2 (containing the BCL2 coding sequence) and the TRE-BCL2-3′UTR (comprising both the BCL2 coding sequence and their regulatory 3′UTR region) vectors contained a green fluorescent protein (GFP) mark (Ubi-GFP).\n",
      "\n",
      "\n",
      "**BCL2 downregulation**\n",
      "\n",
      "Two ShRNAs (shRNAs) targeting BCL2 were designed and cloned into an inducible lentiviral vector (Cellecta) encoding green fluorescent protein (GFP) as a reporter gene and a Tet-On system allowing inducible shRNA expression upon doxycycline treatment (1ug/ml). These shRNA was used to specifically downregulate BCL2 expression in TMD8 and OCILY19 lymphoma cell lines. The efficiency of BCL-2 knockdown was confirmed by qPCR and Western blot analysis. The two shRNA sequences are listed below:\n",
      "\n",
      "BCL2sh1:\n",
      "\n",
      "Fw: ACCGGTGGATGACTGAGTACCTGAACGTTAATATTCATAGCGTTCAGGTACTCAGTCATCCATTTT.\n",
      "\n",
      "Rv: CGAAAAAATGGATGACTGAGTACCTGAACGCTATGAATATTAACGTTCAGGTACTCAGTCATCCAC.\n",
      "\n",
      "BCL2sh2:\n",
      "\n",
      "Fw: ACCGGGTGATGAAGTACATCCATTATGTTAATATTCATAGCATAATGGATGTACTTCATCACTTTT.\n",
      "\n",
      "Rv: CGAAAAAAGTGATGAAGTACATCCATTATGCTATGAATATTAACATAATGGATGTACTTCATCACC.\n",
      "\n",
      "\n",
      "**qPCR assays**\n",
      "\n",
      "Total RNA was extracted with the ZYMO RESEARCH Quick-RNA MiniPrep kit protocol. A total of 1μg of RNA was converted to cDNA using LunaScript RT SuperMix kit (Bio-Labs) following the manufacturer’s protocol. Quantitative polymerase chain reaction (qPCR) was performed using the Fast SYBR Green master mix (Applied biosystems) in a ViiA7 Real-Time PCR System (Bio-Rad). Relative RNA enrichment was calculated using the ΔΔCt method, normalized to B-actin. Specific primers were designed and listed below:\n",
      "\n",
      "MYC = Fw TTCGGGTAGTGGAAAACCAG; Rv CAGCAGCTCGAATTTCTTCC.\n",
      "\n",
      "ACTIN = Fw GAACGGTGGTGTGTCGTTC; Rv GCGTCTCGTCTCGTCTCACT.\n",
      "\n",
      "BCL-2 = Fw GAGTTCGGTGGGGTCATGT; Rv GCCGGTTCAGGTACTCAGTC\n",
      "\n",
      "rRNA45S = Fw GCGTCTCGTCTCGTCTCACT; Rv GAACGGTGGTGTGTCGTTC.\n",
      "\n",
      "TP53 = Fw GAGCTGAATGAGGCCTTGGA; Rv CTGAGTCAGGCC CTTCTGTCTT.\n",
      "\n",
      "CDKN1A (P21) = Fw AAATCGTCCAGCGACCTTCCTCAT; Rv TCTGACTCCTTGTTCCGCTGCTAA.\n",
      "\n",
      "PUMA = Fw TTACTTCCTGCCCTGCTCTGGTTT; Rv TCTCAGGAGGTGCACGTTTCATCA.\n",
      "\n",
      "TBP = Fw CACGAACCACGGCACTGATT; Rv TTTTCTTGCTGCCAGTCTGGAC.\n",
      "\n",
      "\n",
      "**Co-immunoprecipitation (Co-IP)**\n",
      "\n",
      "For immunoprecipitation, cells were lysed on ice in immunoprecipitation buffer (25 mM Tris HCl [pH 7.5], 150 mM KCl, 5 mM MgCl2, 1 mM EGTA, 1mM DTT, 10% glycerol, 0.8% Igepal/NP40, and protease inhibitors PMSF, Leupeptin and Aprotinin). The lysates were cleared by centrifugation and quantified by Bradford protein assay (Bio-Rad). 2 mg of proteins were incubated at 4°C with rotation overnight in immunoprecipitation buffer with anti-MDM2 (SMP14) or Mouse IgG Abs. ProteinA-coated agarose beads (Santa Cruz) were added to the extracts and mixed by rotation for an additional 2h at 4°C. Finally, the beads were washed four times with immunoprecipitation buffer and resuspended in protein loading buffer for western blot analysis. 5% of total lysates was used al INPUT.\n",
      "\n",
      "\n",
      "**Immunoprecipitation-qPCR (IP-qPCR)**\n",
      "\n",
      "Nucleolin immunoprecipitation was performed to suit protein-RNA interaction studies to evaluate the enrichment of BCL2 mRNA associated with nucleolin. Briefly, cells were lysed in PLB buffer (10 mM HEPES, pH 7, 100 mM KCl, 5 mM MgCl2, 0,5% NP40, 1nM DTT, RnaseOut and protease inhibitor cocktail). Following centrifugation, the supernatant was pre-cleared with Protein A Sepharose beads to minimize non-specific binding and quantified. Nucleolin was immunoprecipitated from 2mg lysate by incubating the pre-cleared lysate with 5 μg of anti-Nucleolin antibody (Cell Signaling D4C70) overnight at 4°C with gentle rotation. Immune complexes were then captured by adding Protein A Sepharose beads and incubating for 2 h at 4°C. Beads were washed sequentially with NT2 washing buffer (50 mM Tris, pH 7.4, 150 mM NaCl, 1 mM MgCl2, 0,05% NP40, 0.1nM DTT and RnaseOut) to reduce background signal.\n",
      "\n",
      "Following IP, RNA was extracted from the bead-bound immune complexes using ZYMO RESEARCH Quick-RNA MiniPrep kit according to the manufacturer’s instructions. RNA concentration and purity were assessed using a NanoDrop spectrophotometer. The extracted RNA was reverse transcribed into cDNA using the LunaScript RT SuperMix kit (Bio-Labs) and quantitative PCR (qPCR) was performed using the Fast SYBR Green master mix (Applied biosystems) in a ViiA7 Real-Time PCR System (Bio-Rad). Relative RNA enrichment was calculated using the ΔΔCt method, normalized to input RNA and a negative IgG control.\n",
      "\n",
      "\n",
      "**RNA sequencing**\n",
      "\n",
      "Following pre-treatment with doxycycline 1 μg/ml for 96 h, SUDHL-5 cells transfected with empty vector (Empty) or BCL-2 TET-ON inducible system (BCL-2), were incubated for 6h with Doxorubicin 100nM, Act D 2.5nM, CX-5461 2500nM. Total RNA extraction was performed using ZYMO RESEARCH Quick-RNA MiniPrep kit and libraries prepared using TruSeq RNA Sample Preparation Kit (Illumina) following manufacturer instructions. Bioanalizer 2100 (Agilent Technologies) was used to qualitatively and quantitatively check prepared libraries. Final libraries, prepared in triplicate, were sequenced on an Illumina Hi-Seq 2000with paired-end 2 x 51-bp read lengths.\n",
      "\n",
      "Quality control of the reads was performed using the FastQC software. Adapter removal and trimming was performed using BBDuk-BBMap. The reads were aligned to the reference genome (GRCh38) using STAR software. The STAR –quantMode GeneCount option was used to output a ReadSPerGene count file for each sample. The computed output was then used as the input for the DESeq2 analysis. RNA counts were normalized and used for PCA. To estimate the differential expression, the gene count matrix was analyzed using DESeq2. Upregulated and downregulated genes were selected by setting a false discovery rate lower than 5%. Pathway enrichment analysis using the KEGG database was performed using ClusterProfiler.\n",
      "\n",
      "\n",
      "**In vivo experiment**\n",
      "\n",
      "In vivo studies were performed using immunodeficient NOD.Cg-PrkdcSCIDIl2rgtm1Wjl/SzJ (NSG) mice (Charles River Laboratories, Italy). Subcutaneous tumors were grown in the flanks of mice by injecting 0.3×106 primary tumor cells derived from a DLBCL patient (LNH1). Tumor growth was monitored twice a week using a digital caliper, and tumor volume was calculated according to the formula L × W2/2 (mm3), where W represents the width and L is the length of the tumor mass. Treatments were started approximately 10 days after tumor injection, when a palpable tumor was detected in each mouse.\n",
      "\n",
      "The experimental treatment was performed for 2 weeks as follows: Vehicle, Act D 0.04 mg/kg, venetoclax 50 mg/kg, idasanutlin 100 mg/kg, and the combinations (Act D + venetoclax; Act D + idasanutlin; venetoclax + idasanutlin; Act D + venetoclax + idasanutlin). Treatments were administered five days/week for 14 days, and each group contained five mice. Tumor volume was measured using a caliper every 5 days.\n",
      "\n",
      "Act D was dissolved in dimethyl sulfoxide (DMSO) and diluted with PBS. Venetoclax was prepared weekly in 10% ethanol, 30% polyethyleneglycol-400, and 60% phosal 50 propylene glycol. Idasanutlin was formulated weekly in 60% water, 30% polyethyleneglycol-400, 5% tween-80 and 5% DMSO. The mice were observed daily throughout the treatment period for signs of morbidity/mortality.\n",
      "\n",
      "\n",
      "**In vitro proliferation assay**\n",
      "\n",
      "Cells were seeded in 96-well plates at 25,000cell/100μL/well with either DMSO 0.1% or increasing concentrations/combinations of drugs for 24 and 48 h. Cell viability was assessed by adding Cell Titer Glo reagent (Promega) to the culture medium, according to manufacturer’s instructions.\n",
      "\n",
      "\n",
      "**Caspase 3/7 activity assay**\n",
      "\n",
      "Cells were seeded in 96-well plates at 25,000cell/100μL/well with either vehicle (DMSO 0.1%) or drugs for 12 h. Measurements of caspase activities in cells were performed using the commercially available Caspase-Glo 3/7 Assay (Promega), according to the manufacturer’s instructions.\n",
      "\n",
      "\n",
      "**Western blotting**\n",
      "\n",
      "Preparation of cellular protein lysates was performed by using the Cell Signaling lysis buffer (#9803) according to manufacturer’s extraction protocol. A total of 30 μg of protein was denatured in Laemmli buffer at 95°C for 5 min and western immunoblotting was performed using the Biorad system (TGX 4–15% gels). Transfer was performed using the Trans Blot turbo system (Bio-Rad) onto Nitrocellulose membranes. Images were acquired by using the BioRad Imaging Chemidoc MP system. The antibodies used are listed in the key resources table.\n",
      "\n",
      "\n",
      "**Cell cycle analysis**\n",
      "\n",
      "Single cell suspensions were prepared and fixed with 70% fresh Ethanol for 2 h. After fixation, cells were washed twice with 1x PBS, 1% BSA. Cells were then stained with Propidium Iodide (PI Sigma) 10 μg/ml + RNase overnight, protected from light before analysis. Data Acquisition: Flow cytometric data were acquired on a BD FACS Celesta using BD FacsDIVA Software. Analysis was performed using Flow.Jo 9.3.\n",
      "\n",
      "\n",
      "**Nucleolin immunofluorescence staining and confocal laser microscopy**\n",
      "\n",
      "500.000 SUDHL5 Empty or BCL2 cells were fixed on polylysined slides using 4% paraformaldehyde for 20 min at room temperature. To detect C23 protein, cells were treated with Triton X-100 0,5% (Sigma-Aldrich) and washed with phosphate buffered saline (PBS). After 30min of blocking in PBS +2% BSA, anti-Nucleolin 4E2 mouse antibody, 1/400 (GeneTex) and anti-BCL2 E17 rabbit antibody 1/100 (Abcam) were added, in a dark, humidified chamber (30 min; 37°C). After five washes, cells were incubated with secondary antibodies: anti-mouse Alexa Fluor 647 and anti-rabbit Alexa Fluor 488 1/400 (40 min; 37°C). After five washes, cells were counterstained with DAPI 1/5000 (30 min; 37°C) and mounted in Mowiol.\n",
      "\n",
      "Images were obtained using an SP5 AOBS confocal microscope equipped with a 63X/1.4NA oil-immersion objective, PMT and HyD detectors and LasX 3.5.5 acquisition software (Leica Microsystems GmbH). Images were processed using Fiji/ImageJ software (v1.52, National Institutes of Health). Images represent a single confocal plane where brightness and contrast were optimized.\n",
      "\n",
      "\n",
      "**Immunohistochemistry**\n",
      "\n",
      "Immunohistochemistry (IHC) was performed on tissue microarrays (TMAs) prepared from paraffin blocks using a precision instrument as previously described. The following primary antibodies were used: anti-BCL2 (clone 124, Dako Agilent, dilution 1:100), anti-cMYC (clone EP121, Epitomics, dilution 1:100), TP53 (clone DO-7, Dako Agilent, dilution 1:150) and anti-Nucleolin (clone 4E2, Abcam, dilution 1:6000). For BCL2, cMYC and TP53 antigen retrieval was carried on PT-links at 92°C for 5 min in EnVision Flex Target Retrieval Solution High pH (Dako Agilent, Glostrup, Denmark). IHC tests were performed on AutoStainer Plus platforms, using the LSAB+REAL Detection System (Dako Agilent). The IHC preparations were counterstained with Gill’s hematoxylin and mounted in Kaiser’s glycerin. The cut-off values of 40%, 50% and 50% positive neoplastic cells were applied for MYC, BCL2 and TP53 respectively.56 Nucleolin antigen retrival was carried out in Tris-HCL Buffer pH 8,5, at 95°C for 40 min and IHC test in Benchmark Ultra immunostainer using OptiView DAB detection kit (brown color) (Ventana Diagnostic Systems, Tucson, AZ, USA). Sections were counterstained using Hematoxylin and Bluing reagent following Ventana indications. Three different nucleolin expression patterns were identified: pattern 1, in which Nucleolin appeared to be exclusively located in the nucleolus; pattern 2, in which the staining reaction, other than in the nucleolus, occurred in the nucleoplasm and sometimes homogeneously stains all the nucleus; pattern 3, in which the nuclei of all the cells were uniformly stained, thus completely masking the visualization of the nucleolus.\n",
      "\n",
      "\n",
      "**Nucleolar organizer regions silver staining and image cytometry**\n",
      "\n",
      "After silver staining the NORs are identified as black dots exclusively localized throughout the nucleolar area. NOR silver staining was performed according to the guidelines of the “International Committee on AgNOR quantitation”.57 Briefly, slides were moved from water to heat-resistant plastic Coplin jars, fully immersed in 10mM sodium citrate buffer (pH 6.0) and autoclaved at 120°C for 20 min. After cooling to room temperature in the sodium citrate buffer, slides were stained with silver for 13 min at 37°C in the dark using a solution of one volume 2% gelatine in 1% aqueous formic acid and two volumes of 50% silver nitrate. Sections were finally dehydrated and mounted in Canada balsam without any counterstaining.\n",
      "\n",
      "Morphometric analysis of AgNORs was carried out using the Image-Pro Plus program (Media Cybernetics, Silver Spring, Maryland, USA). The main stages of image processing were as follows: a microscope field was selected by the operator using a ×40 objective lens. The selected image was then captured and stored in the digital memory, and displayed on the color monitor. Here, the operator interactively defined the gray threshold, which permitted automatic quantification of the black dots corresponding to the silver-stained nucleoli. The morphometric analysis was then performed on a cell-by-cell basis by converging the window over the corresponding nucleus. For each case, the AgNOR area of at least 200 nuclei was measured and the mean (SD) AgNOR area was calculated.\n",
      "\n",
      "\n",
      "**FISH analysis**\n",
      "\n",
      "MYC, BCL2 and BCL6 translocations were tested by FISH. FISH studies were conducted on paraffin sections using the following probes: (Abbott, Abbott Park, IL, USA) Vysis LSI MYC dual color break-apart, Vysis LSI BCL2 dual color break-apart, Vysis LSI BCL6 dual color break-apart. Slides were deparaffinized, co-denatured with probe, hybridized overnight, washed and then mounted with DAPI/Antifade. For each probe, 200 interphase nuclei were analyzed to detect re-arrangements and numerical abnormalities. Cut-off values were established for each probe by assessing 10 normal controls (reactive lymph nodes) and choosing values three standard deviations above the mean.\n",
      "\n",
      "\n",
      "**Targeted gene expression profiling**\n",
      "\n",
      "Digital quantification of gene expression was performed using the NanoString platform. Total RNA was extracted from three 15μm sections of each FFPE sample using the RecoverAll Total Nucleic Acid Isolation Kit for FFPE (Invitrogen). The yield of the extracted RNA was assessed using a NanoDrop ND-1000 Spectrophotometer (NanoDrop Technologies). RNA quality was scored according to the DV200 value (% of RNA fragments >200 nucleotides) using the Agilent RNA 6000 Nano Chip Kit (BioAnalyzer, Agilent Technologies). Only samples with a DV200 value > 20% passed the QC step.\n",
      "\n",
      "Two custom panels were profiled to determine the Cell-of-Origin molecular subtype and to quantify the expression of genes belonging to the BCL-2 family. The COO/MYC-BCL2 panel contained 22 genes: 20 genes for COO subtyping plus MYC and BCL-2.58 To determine the COO molecular subtype, expression data were analyzed at NanoString Technologies, Inc. using a modified RUO version of the NanoString Lymphoma Subtyping Test (LST) algorithm.13,59,60\n",
      "\n",
      "The LST algorithm measures the geometric mean of 5 housekeeping genes (HK geomean) to ensure RNA quality based on a pre-defined clinical QC threshold of 128 (indicated as Pass). An HK geomean value below 64 is deemed insufficient RNA quality to provide a subtyping result (listed as fail). A value between 64 and 128 is “borderline” quality since it meets previously published thresholds for RNA quality within clinical research studies but it does not meet Nanostring clinical QC threshold of 128 for individual patients.\n",
      "\n",
      "Each sample meeting the QC threshold was assigned to one molecular subtype: Activated-B-Cell (ABC), GC-B-Cell (GCB) or Unclassified within an equivocal zone.\n",
      "\n",
      "The quality control and normalization of NanoString nCounter were performed using R package NanoStringNorm. The raw NanoString counts for each gene were subjected to a technical normalization considering positive and negative probes. A normalization factor was calculated by obtaining the geometric mean of the positive controls used for each sample and applied to the raw counts of the nCounter output data to eliminate variability unrelated to the samples. The resulting data were normalized again with the geometric mean of the housekeeping genes. Normalized data were log2-transformed for further analyses.\n",
      "\n",
      "\n",
      "**RNA-seq data analysis**\n",
      "\n",
      "Gene expression data were derived using an automated pipeline based on anaconda environment.61 The analysis steps are listed in sequence.\n",
      "\n",
      "\n",
      "**Quality control**\n",
      "\n",
      "Quality check is performed using FastQC v0.11.9 and it is run for every sequence file independently\n",
      "\n",
      "fastqc --o output_folder read1.fastq.gz read2.fast.gz.\n",
      "\n",
      "\n",
      "**Quality trimming and adapter removal**\n",
      "\n",
      "The sequence files are processed with bbduk v38.79 to remove parts of the readouts that match adapter sequences and to remove poor quality base pair readouts at the end of the reads.\n",
      "\n",
      "bbduk.sh in1 = read1.fastq.gz in2 = read2.fastq.gz out1 = read1_trim.fastq.gz out2 = read2_trim.fastq.gz qtrim = r trimq = 20 minlen = 25.\n",
      "\n",
      "bbduk.sh in1 = read1_trim.fastq.gz in2 = read2_trim.fastq.gz out1 = read1_cleaning.fastq.gz out2 = read2_cleaning.fastq.gz ref = adapters.fa ktrim = r k = 23 mink = 11 hdist = 1 tpe tbo.\n",
      "\n",
      "\n",
      "**Generate genome indexes**\n",
      "\n",
      "The GRCh38 human genome assembly is used as the genome target and the transcriptome target is defined by the GENCODE release 34 (basic gene annotation). Genome indexing is performed using the following command.\n",
      "\n",
      "STAR --runThreadN 8 --runMode genomeGenerate –genomeDir genome_folder --genomeFastaFiles GRCh38.primary_assembly.genome.fa --sjdbGTFfile gencode.v34.primary_assembly.annotation.gtf --sjdbOverhang 50.\n",
      "\n",
      "\n",
      "**Mapping and quantification**\n",
      "\n",
      "Reads mapping was performed using STAR v2.7.3a. The STAR -quantMode GeneCount option was used for gene quantification, with the resulting strand counts chosen depending upon library preparation (unstranded).\n",
      "\n",
      "STAR –runThreadN 8 --runMode genomeGenerate --genomeDir indices_folder –genomeFastaFiles GRCh38.primary_assembly.genome.fa --sjdbGTFfile gencode.v34.primary_assembly.annotation.gtf --sjdbOverhang 50.\n",
      "\n",
      "\n",
      "**Differential expression analysis**\n",
      "\n",
      "STAR read counts were used as input into DESeq2.62 Genes with raw counts greater than 20 were kept for subsequent differential expression gene (DEG) analyses. Mitochondrial genes were removed from the downstream analysis. DEG analysis was performed with drug inhibiting ribosome biogenesis (doxorubicin, Act D, CX-5461) and BCL-2 status (presence or absence) as the covariates of interest and DEG were called based upon a false discovery rate less than 0.05 (Benjamini-Hochberg correction). Raw counts were normalized using DESeq2’s function regularized transformation (rlog), and normalized counts were used to perform and visualize PCA results.\n",
      "\n",
      "\n",
      "**Kyoto Encyclopedia of Gene and Genome pathway analysis of differential expressed genes**\n",
      "\n",
      "To better understand the gene functions, the KEGG enrichment analysis of the DEGs was performed using clusterProfiler package.63,64 We first converted the gene ID from the Official Symbol of the differential genes obtained by org.Hs.e.g.,.Db package using clusterProfiler’s function ‘bitr’ followed by a KEGG analysis. Adjusted p-value (Benjamini-Hochberg method) has been used for terms ranking and selection (padj<0.05).\n",
      "\n",
      "\n",
      "**p53 targets: Identification and fold changes evaluation**\n",
      "\n",
      "In order to quantify the p53-mediated response after treatment with RBi in SUDHL-5 cells, we merged the six different DEGs lists in a consensus list and queried TRANSFAC database containing a list of direct p53 targets. Starting from these direct p53 targets (n = 256), we applied a two-step filter.(1)only those genes with log2\n",
      "|fc| ≥ 1 in DMSO were considered(2)only those genes with |Δfc| = |fcEmpty\n",
      "- fcBCL-2\n",
      "|≥0.58 (where fc is expressed in log2) in each treatment arm were selected\n",
      "\n",
      "The two-step filter allowed to highlight only the significantly regulated p53 targets. At this point, levels of fold change (log2) of significantly regulated p53 targets identified in SUDHL-5 cells treated with RBi in presence versus absence of BCL-2 were visualized using radar charts.\n",
      "\n",
      "\n",
      "**Quantification and statistical analysis**\n",
      "\n",
      "Quantitative data were collected from at least three independent experiments and are expressed as mean ± SD. Student’s t test was used to assessed significant differences between groups. The statistical details are described in the figures and in their corresponding legends for individual experiments.\n",
      "\n",
      "For survival analysis, we used the Kaplan-Meier method to estimate OS and PFS. A log rank test assessed significant differences between curves, and the threshold to stratify patients into low- and high-risk groups was determined using the median value. Multivariate and univariate analyses were performed using the Cox proportional hazards regression model. Associations and differences in patient characteristics were analyzed using the χ2 and Fisher’s exact test. p ≤ 0.05 was considered statistically significant.\n",
      "\n",
      "\n",
      "**Additional resources**\n",
      "\n",
      "The 46 patients from the DLCL04 study,39 were enrolled in the ClinicalTrials.gov, number NCT00499018 (https://clinicaltrials.gov/expert-search?term=NCT00499018).\n",
      "---------------------------------\n",
      "PMC ID: PMC12018228\n",
      "\n",
      "**Introduction**\n",
      "\n",
      "Atopic dermatitis (AD) is a complex and heterogeneous systemic disease (1, 2) and the most common chronic skin disorder (3), that affects a large number of people at any age (4). The heterogeneity of AD lies both at pathophysiology and clinical levels (5), leading to a classification into different phenotypes or endotypes (6).\n",
      "\n",
      "Clinical and disease characteristics of AD allow for a stratification into different disease phenotypes based on age, age of onset, ethnicity, and presence of other atopic diseases, such as allergic rhinitis and asthma (5, 7, 8). However, a more recent categorization, based on the underlying molecular mechanisms known as endotypes, has been proposed (9).\n",
      "\n",
      "The high complexity and heterogeneity of these molecular mechanisms in AD, also determining the different clinical phenotypes, might explain the different responses to treatments among patients (5, 10). Although endotypes could guarantee a more efficient classification of AD, the differences in the immune profile across patient groups have not been fully elucidated yet (9). Moreover, the definition of the immune profile (or patients’ endotypes) and the identification of a panel of multiple biomarkers (11) may allow patient clustering into subgroups, and thus the tailoring of the therapeutic approach through the use of more targeted therapeutic approaches (5).\n",
      "\n",
      "The recent introduction of targeted biologic and systemic Janus kinase (JAK) inhibitors has profoundly changed the therapeutic landscape in AD (12), offering the possibility of more personalized approaches based on patient-specific biomarkers, which are defined as measurable indicators of some biological state or condition evaluated on different biological materials, including whole blood, plasma, serum, and tissue samples, on which molecular, genetic, or morphologic data can be analyzed (10).\n",
      "\n",
      "Biomarkers, when assessed early, may predict therapeutic responses to drugs and, therefore, identify patient subsets that would benefit most from a treatment (10, 11). However, nowadays, AD is still a clinical diagnosis based on physicians’ clinical experience and experts agree on the unmet need to identify a list of reliable biomarkers useful for patients’ stratification and treatment monitoring (13).\n",
      "\n",
      "The state-of-the art knowledge about biomarkers in AD was recently reviewed by Renert-Yuval et al. (10), Bakker et al. (11), and Park et al. (14). Here, we discuss the most up-to-date knowledge about predictive biomarkers. For this purpose, we selected the most clinically impactful papers (including original articles, narrative, and systematic reviews, expert consensus, and expert opinions) published on this topic in journals indexed by PubMed, between 2016 and 2024.\n",
      "\n",
      "\n",
      "**Biomarkers by treatment options**\n",
      "\n",
      "Although clear suggestions for clinical practice are still lacking, several biomarkers aiming at predicting a therapeutic response in patients with AD have been studied.\n",
      "\n",
      "\n",
      "**CCL22**\n",
      "\n",
      "Glickman and colleagues compared clinical and molecular outcomes of different treatments for AD and showed that baseline CCL22 expression was correlated with clinical improvement across multiple studies at various time points without being drug-specific (15).\n",
      "\n",
      "\n",
      "**CXCL2**\n",
      "\n",
      "The baseline levels of Th17 cell-related cytokine CXCL2 predicted response to therapy at week 16 in patients treated with dupilumab which targets the IL-4 receptor alpha chain that inhibits both IL-4 and IL-13-related cascades (15). However, a clear molecular explanation of this association is lacking and no further studies have corroborated the role of CXCL2 as biomarker yet.\n",
      "\n",
      "\n",
      "**IgE**\n",
      "\n",
      "Based on the IgE levels and IgE specific to protein antigens, AD was classified into extrinsic and intrinsic subtypes: the first characterized by high IgE and type 2 cytokines levels, and the second by normal IgE and lower type 2 cytokines levels (9). Measurement of IgE levels has been a routine practice in diagnosing AD for a long time, possibly because IgE levels could be high in the extrinsic AD form and clinicians often consider the serum total IgE levels as a biomarker of AD severity (10). Despite the historical association between high IgE levels and type 2 cytokines pathway, patients’ responses to dupilumab were not clearly correlated to baseline IgE levels (10), while changes in the IgE levels during the course of treatment might be a tool to monitor therapeutic efficacy according to other evidence (16). Also, no significant changes in total and specific IgE levels were observed after 16 weeks of upadacitinib treatment during a phase 2 study, while still proving its efficacy at clinical level (17). Hence, IgE levels do not appear to be a candidate for predictive biomarkers.\n",
      "\n",
      "\n",
      "**Dipeptidyl peptidase-4 (DPP-4) and periostin**\n",
      "\n",
      "Baseline high levels of serum of DPP-4 and periostin, indicators of increased activity of IL-13, have been identified as potential biomarkers of a good response to tralokinumab, a fully human mAb blocking IL-13 (18). In an exploratory analysis of a phase 2b study, Wollenberg and colleagues observed that in patients with moderate-to-severe AD and a baseline higher concentration of PPD-4 and periostin, tralokinumab led to greater changes from the baseline on the Eczema Area Severity Index (EASI) (18).\n",
      "\n",
      "\n",
      "**IL5RA, CCL23, and eosinophil count**\n",
      "\n",
      "Upadacitinib, an oral Janus kinase (JAK)-1 inhibitor significantly reduced, among other mediators, the levels of serum IL5RA and CCL23 as early as week 1 of treatment (19). A significant change was also a reduction in absolute eosinophils count after week 2 of upadacitinib treatment, maintained through week 24 (19). Hagino et al. showed that total eosinophil count correlates with EASI and PP-NRS during upadacitinib treatment over 48 weeks (20), indicating its potential as a biomarker reflecting treatment responses. IL5RA and CCL23 genes were upregulated in the eosinophil-high endotype (21). It could be interesting to assess whether patients with a high-eosinophil profile might benefit the most from treatment with upadacitinib. In addition, a real-world study from Zheng et al. showed significant eosinophil count reduction after 4 weeks of abrocitinib treatment in 47 patients (22).\n",
      "\n",
      "\n",
      "**IL-22**\n",
      "\n",
      "According to Brunner et al. baseline IL-22 expression can be a biomarker able to predict a therapeutic response to fezakinumab. Results from a phase 2a clinical trial showed that among patients stratified based on baseline skin IL-22 mRNA expression, those with high IL-22 baseline levels were more likely to improve both at clinical and transcriptomic level. Fezakinumab did not meet the primary endpoint of the study at week 12 (23). However, also JAKi targets IL-22: IL-22-attracting chemokines were significantly reduced with upadacitinib treatment as early as week 2, suggesting that upadacitinib may have early and robust effects on Th22 axes (17). Also abrocitinib reduced Th22-associated genes (S100A8/9/12) in a dose-dependent manner, starting from week 2 throughout week 12 (24).\n",
      "\n",
      "\n",
      "**The role of AD pathogenetic pathways in predicting response**\n",
      "\n",
      "Four differentiated clusters of patients have been identified by using a data-driven approach based on serum biomarkers (6). Each cluster was characterized by a specific serum biomarker profile, implying that a distinct underlying immunopathologic pathway drives each cluster. Two of these clusters were characterized by a Th2-dominated biomarker profile (i.e., “Th1/Th2/Th17-dominant” and “Th2/Th22/PARC-dominant”), with high levels of Th2-related cytokines, suggesting that patients in these clusters would be ideal candidates for Th2-blockers (6). An exploratory analysis of the phase 2b study with tralokinumab found a better treatment response in patients with high levels of DPP-4 and periostin (18). Pediatric AD is characterized by a Th2, Th9, and Th17 higher polarization, thereby suggesting that the current biologic drugs targeting Th2 cytokines would be more effective in children than in adults, who, instead, have a more pronounced Th22 dominant T-cell response (5). Similarly, the Th17 higher contribution reported in the Asian population would support an effect of anti-IL-17 therapy, such as secukinumab, in order to treat AD in this ethnic group (5, 25). However, a phase 2b randomized double-blind controlled study failed to show a clinical benefit of secukinumab in adult AD, and several cases of anti-IL-17-induced AD have been reported (25). Therefore, the IL-17 pathway, although may contribute, does not appear to be essential to AD pathogenesis. Indeed, it is well known that together with Th2 and Th22 inflammatory pathways, the involvement of Th1 and Th17 play also a role in AD and varies by patient race and age (24). Upadacitinib, beyond the clinical efficacy, downregulated key AD genes associated of multiple pathways such as Th2 (CCL17), Th1 (CXCL10), and Th22 (S100A9/12, PI3) potentially supporting a more complete disease control (17). Abrocitinib showed similar results, downregulating genes associated with inflammation, epidermal hyperplasia, Th2 and Th22 immune responses in the skin of patients with moderate-to-severe AD (24).\n",
      "\n",
      "\n",
      "**Sources of biomarkers**\n",
      "\n",
      "Biomarkers differ according to the source of biological material: (i) genomic information, (ii) transcriptomic profiles, (iii) proteins from body fluids or tape stripping, and (iv) morphological information (10), with possible sources of biomarkers being blood, tape strips of skin, and skin biopsy (26). Skin biopsy, although invasive, accurately reflects disease severity and often detects AD biomarkers not measurable by blood tests. The tape strip sampling technique, a non-invasive approach that collects stratum corneum proteins, has shown promising results (10, 27, 28). Not all biomarkers discussed here can be measured in both serum and skin (10). A minimally invasive approach guarantees repeated assessments of biomarkers and would be ideal to define the therapeutic outcome and improve patients’ adherence.\n",
      "\n",
      "\n",
      "**Clinical predictive factors**\n",
      "\n",
      "Given the current landscape of AD, reliable biomarkers are necessary; hence, if blood-based or skin-based molecular markers will not bring these tools to clinical practice, then physicians could leverage on clinical predictive factors, based on patients’ signs, symptoms, and comorbidities (29). A recent post hoc analysis showed that baseline body surface area affecting 10–40% and severe itch (i.e., Numerical Rating Scale ≥7) were the strongest predictors of the response to the oral JAK 1/2 inhibitor baricitinib, in adults with moderate-to-severe AD (30). Likewise, a retrospective study on patients with moderate-to-severe AD, showed a greater likelihood of response to upadacitinib 15 mg in those with lower EASI (<24) and older age and to upadacitinib 30 mg for individuals with lower levels of IgE and LDH (31). AD localization in exposed areas at the baseline and AD persistency in the head/neck, a known phenomenon in difficult-to-treat AD patients, may also drive treatment choice (32). Furthermore, de novo appearance of dupilumab-associated head and neck dermatitis, now recognized as a distinct entity, can be successfully resolved with upadacitinib (33) and seems to be associated with normalization of IL-4/IL-13 downstream activity markers such as CCL13, CCL17, CCL18, and CCL26, but also with a strong increase in type 22-associated inflammation, enhanced keratinocyte activation, and IL-22 receptor upregulation (34).\n",
      "\n",
      "\n",
      "**The relevance of predictive biomarkers in daily practice**\n",
      "\n",
      "The tailorization of the therapeutic approach and predictable response of each drug (or class of agents) for a certain patient subtype, would be impactful because it could help lowering the economic burden of targeted therapies using those that likely could be more successful for that specific endotype. The identification of patients potentially responding to one selective agent could increase treatment compliance avoiding switching a time prior to achieve a valid and satisfactory response.\n",
      "\n",
      "\n",
      "**Conclusions and future outlook**\n",
      "\n",
      "The present therapeutic scenario for AD, including the newly developed biologic and targeted agents, highlights the importance of overcoming the “one-size-fits-all” approach toward a precision medicine one, especially for such a multifactorial and complex disease like AD. There is a need for the identification and validation of reliable biomarkers to predict and monitor the treatment response as objective tools in daily clinical practice: Real-world studies in which single biomarkers or biomarkers profiles (i.e., determined by clusters of biomarkers) are measured in relationship with clinical scoring systems are lacking. Here, we have described few example of candidate biomarkers that might predict a response to treatments. However, no one is currently applied in routine clinical settings. A possible limitation to this hard search for valid biomarkers could be found in the lack of stratification of the studied populations into the responder/non-responder.\n",
      "\n",
      "Our work wants to draw the attention of the scientific community on this theme, ultimately aiming at improving AD management and advancing the standard of care. Further research will be pivotal for clarifying whether molecular biomarkers of therapeutic response will ever represent a practical tool assisting dermatologist in the selection of the best treatments in patients with AD.\n",
      "---------------------------------\n",
      "PMC ID: PMC12018088\n",
      "Research in contextEvidence before this studyChronic kidney disease (CKD) is a non-communicable disease of global public concern with an increasing prevalence. CKD progression and cardiovascular complications can be attenuated with disease monitoring and guideline-directed care. We performed a literature review (across EMBASE Ovid and PubMed) of observational studies relating to patterns of CKD care. Internationally, variability in the concordance of CKD care with clinical practice guidelines has been reported, with gaps in monitoring and use of guideline recommended therapies. In Australia, however, such assessments have been limited.Added value of this studyUsing a large nationally representative Australian primary care data source, we outline the CKD monitoring and cardiovascular risk management patterns, from 2011 to 2022. Our study found low rates of completion of monitoring (34%) largely driven by low albuminuria monitoring. We have identified several priority areas for improving CKD care including increasing albuminuria monitoring and improving cardiovascular risk management. Our findings highlight the need for implementation strategies to improve CKD care.Implications of all the available evidenceCKD is a global public health concern. There are significant gaps in the identification, monitoring and management of CKD internationally and in Australia. Future work is required to further understand facilitators and barriers of CKD care in primary care, with a subsequent focus on implementation strategies to improve CKD care.\n",
      "\n",
      "\n",
      "**Introduction**\n",
      "\n",
      "Chronic kidney disease (CKD) is a global public health concern, estimated to affect greater than 10% of the world's population,1 and this burden of CKD continues to increase.2 Without fundamental improvements in prevention and management, CKD is estimated to become the fifth leading cause of years of life loss by 2040.3\n",
      "\n",
      "In Australia the prevalence of CKD is estimated to be up to 11%,4,5 representing up to 2.8 million people. Much of the morbidity and mortality of CKD is attributed to cardiovascular disease or progression to advanced kidney disease. Routine monitoring in CKD is essential for risk assessment, detection of deterioration and implementation of guideline directed therapies to improve kidney and cardiovascular outcomes.6\n",
      "\n",
      "CKD is primarily diagnosed and managed in primary care. National primary care guidelines recommend routine surveillance of renal function, albuminuria and blood pressure, in addition to use of angiotensin converting enzyme inhibitors (ACE inhibitor) or angiotensin receptor blockers (ARBs), sodium-glucose co-transporter 2 inhibitor (SGLT2 inhibitor) and statin therapy in those at high cardiovascular risk.7 Emerging international studies on the patterns of CKD management in general practice indicate variable concordance of CKD care with clinical practice guidelines, including substantial gaps in monitoring and use of guideline-recommended therapies.8, 9, 10 In Australia, such assessments have been limited. Recent studies11,12 are limited by their relatively small sample size and geographical coverage,12 and lack of assessment regarding guideline-concordant CKD care in relation to cardiovascular risk management.11\n",
      "\n",
      "Thus, using a large, nationally representative, primary care data source in Australia, we conducted a community-based cohort study to assess CKD monitoring practices (measurement of cardio-renal-metabolic parameters) and cardiovascular risk management (blood pressure and lipid control), and their alignment with clinical guideline recommendations for CKD care.\n",
      "\n",
      "\n",
      "**Methods**\n",
      "\n",
      "\n",
      "**Study design and source population**\n",
      "\n",
      "MedicineInsight is a national Australian primary care data source that includes de-identified longitudinal electronic health records on patients visiting primary care participating practices across all states and territories.13,14 Recorded data includes information on sociodemographic and clinical characteristics (including comorbidities), laboratory results, and prescription medications.\n",
      "\n",
      "\n",
      "**Identification of study cohort**\n",
      "\n",
      "We derived a study cohort that was comprised of all adults (≥18 years) who had at least 1 clinical encounter at a MedicineInsight participating primary care centre (n = 392 general practices) with ≥1 eGFR measurement (with or without a urine albumin-creatinine ratio; UACR) between 1 January 2011 and 30 June 2020 (Supplementary Figure S1). The index date was defined as when criteria for presence of CKD were met. CKD was defined using the Kidney Disease Improving Global Outcomes (KDIGO) definition,15 with local sex-specific albuminuria criteria as per Australian guidelines16: two consecutive eGFR measures <60 mL/min/1.73 m2 at least 90 days part and/or two consecutive UACR measures ≥2.5 mg/mmol for males and ≥3.5 mg/mmol for females at least 90 days apart. In order to minimise potential misclassification of CKD status, due to the use of different eGFR equations during the study period, we re-calculated eGFR directly using the 2009 Chronic Kidney Disease Epidemiology Collaboration equation.17 CKD stages were defined as per KDIGO defintions.6\n",
      "\n",
      "\n",
      "**Outcomes**\n",
      "\n",
      "Outcomes were assessed across two domains: (1) CKD monitoring and (2) cardiovascular risk management. Kidney disease guidelines recommend comprehensive CKD monitoring of kidney function, albuminuria, other laboratory parameters (urea and electrolytes, lipids and glycated haemoglobin [HbA1c] in patients with diabetes) and blood pressure monitoring at least yearly in all patients with CKD.15,16 For domain one we defined “core monitoring” as having a record of at least one of all of the following measurements: blood pressure, eGFR, UACR, lipids and HbA1c (in patients with diabetes) within 18 months following the index date (to allow an additional 6 months of potential delays in patient attendance). Individual components of the core monitoring outcome were also assessed separately.\n",
      "\n",
      "For domain two, cardiovascular risk management was defined as, (1) receiving at least one prescription of an angiotensin converting enzyme (ACE) inhibitor or angiotensin receptor blocker (ARB), and statin prescription in patients over 50 years old or under 50 with additional cardiovascular risk factors (i.e., established cardiovascular disease, diabetes and/or stroke), (2) achieving blood pressure targets of <140/90 mmHg and <130/80 mmHg, (3) achieving a low density lipoprotein cholesterol (LDL) < 2 mmol/L, within 18 months following the index date.\n",
      "\n",
      "\n",
      "**Patient characteristics**\n",
      "\n",
      "We assessed baseline patient information including sex, age categories, remoteness (major cities, regional, remote), socioeconomic disadvantage (Index of Relative Socioeconomic Advantage and Disadvantage; defined based on Australian Census data),18 smoking status, eGFR categories, albuminuria categories, and comorbid conditions (atrial fibrillation, cardiovascular disease, heart failure, diabetes mellitus, and hypertension) from the relevant data files in MedicineInsight within one year prior to index date.\n",
      "\n",
      "\n",
      "**Statistical analysis**\n",
      "\n",
      "We calculated the percentage of patients who met the criteria for completion of core monitoring (overall and by subgroups of patients stratified by hypertension status, diabetes status, year of cohort entry, sex and age category) and percentage of patients prescribed ACE inhibitors or ARBs and statins. Percentages were also computed for each individual component of the core monitoring outcome. Chi-square tests were used to determine statistically significant differences in monitoring between subgroups.\n",
      "\n",
      "Modified Poisson models, adjusting for patient characteristics, were constructed to determine variables associated with completion of core monitoring and medication use. Patient characteristics included age, sex, remoteness, socio-economic disadvantage, smoking status, eGFR categories, albuminuria categories and comorbidities (atrial fibrillation, cardiovascular disease, heart failure, diabetes mellitus, and hypertension), and were chosen based on review of prior literature. All candidate variables were forced to be included in the model without any prior selection process. To test for robustness of findings, a sensitivity analysis assessed the completion of core monitoring over a shorter follow-up period (12 months). Additionally, a sensitivity analysis with multiple imputation by fully condition specification19 for missing values was conducted, and Modified Poisson models were repeated to assess patient characteristics associated with core monitoring and cardiovascular risk management. A two-sided p-value <0.05 was considered statistically significant. All statistical analyses were performed with Stata version 17.0 (Stata, TX).\n",
      "\n",
      "\n",
      "**Patient and public involvement**\n",
      "\n",
      "This study was conducted using previously collected, de-identified, longitudinal healthcare data. No patients were involved in setting the research question or the outcome measures, nor were they involved in developing plans to design or implement the study.\n",
      "\n",
      "\n",
      "**Ethical approval**\n",
      "\n",
      "This study was approved by the MedicineInsight Data Governance Committee (2020-004) and Research Ethics Review Committee of the Sydney Local Health District, NSW, Australia (X21-0428, 2020/ETH00963). A waiver of consent was granted for use of non-re-identifiable routinely collected patient data. The study adheres to the RECORD extension of the STROBE reporting guidelines.20\n",
      "\n",
      "\n",
      "**Role of funding source**\n",
      "\n",
      "This study was conducted as part of a research program supported by the University of New South Wales Scientia Program and a sponsorship provided by Boehringer Ingelheim and Eli Lilly Alliance. The design, analysis, interpretation, writing of this manuscript and decision to submit for publication was performed independent of all funding bodies.\n",
      "\n",
      "\n",
      "**Results**\n",
      "\n",
      "\n",
      "**Cohort characteristics**\n",
      "\n",
      "Of the 2,466,293 patients with at least 1 serum creatinine measurement recorded in MedicineInsight during the study period, 140,780 patients (5.7%) were identified as having CKD. Patients with CKD had a mean age of 74.3 years (standard deviation [SD] 12.5), 51.4% were female and 32.4% had a record of diabetes mellitus (Table 1). The mean eGFR was 52.4 mL/min/1.73 m2 (SD 17.5), such that 13.9% of the cohort had stage 1 or 2 CKD, 57.6% had stage 3a CKD, 21.3% had stage 3b CKD and 7.0% had stage 4 or 5 CKD. Almost two-thirds (61.8%) of the cohort did not have a record of UACR measurement at baseline. In patients with both an eGFR and UACR measure (31% of total cohort), enabling risk stratification according to local and international CKD guidelines, 61.2% were at moderate risk, 23.7% high risk and 15.0% very high risk of CKD progression (Supplementary Figure S2).Table 1Baseline characteristics of the study cohort, overall and by eGFR category.TotaleGFR category≥6045–5930–44<30n = 140,780 (%)18,275 (13.9)75,405 (57.6)27,929 (21.3)9172 (7.0)Sociodemographic information Female72,474 (51.4)6378 (34.9)41,775 (55.4)15,952 (57.1)4977 (54.2) Age, yr, mean, (SD)74.3 (12.5)63.7 (13.6)76.2 (10.0)79.0 (10.4)76.9 (13.8) Age category (years) 18–29668 (0.4)312 (1.7)81 (0.1)42 (0.1)60 (0.6) 30–391794 (1.2)796 (4.3)228 (0.3)114 (0.4)147 (1.6) 40–494191 (2.9)1782 (9.7)809 (1.0)284 (1.0)292 (3.1) 50–5910,268 (7.2)3560 (19.4)3416 (4.5)874 (3.1)583 (6.3) 60–6926,274 (18.6)5349 (29.2)13,705 (18.1)3410 (12.2)1178 (12.8) 70–7946,530 (33.0)4604 (25.1)28,487 (37.7)8602 (30.8)2340 (25.5) ≥8051,055 (36.2)1872 (10.2)28,679 (38.0)14,603 (52.2)4572 (49.8) Remotenessa Major cities77,496 (55.3)10,208 (56.1)40,965 (54.5)15,403 (55.4)5128 (56.1) Regional61,366 (43.8)7765 (42.7)33,601 (44.7)12,211 (43.9)3930 (43.0) Remote1234 (0.8)194 (1.0)492 (0.6)169 (0.6)68 (0.7) IRSAD Quintileb 1–2 (most disadvantaged)31,118 (22.2)4327 (23.8)16,599 (22.1)6216 (22.3)2027 (22.2) 3–432,328 (23.0)4055 (22.3)17,518 (23.3)6543 (23.5)2024 (22.1) 5–629,547 (21.0)3908 (21.5)15,517 (20.6)5656 (20.3)1973 (21.6) 7–822,064 (15.7)3052 (16.8)11,752 (15.6)4189 (15.0)1382 (15.1) 9–1025,065 (17.8)2825 (15.5)13,690 (18.2)5185 (18.6)1722 (18.8) Smoking status Smoker8866 (6.7)2416 (13.6)3478 (4.9)1239 (4.9)482 (6.0) Previous smoker52,124 (39.7)7261 (41.1)27,828 (39.3)10,056 (39.8)3136 (39.3) Non-smoker70,165 (53.5)7986 (45.2)39,379 (55.7)13,920 (55.2)4359 (54.6)Laboratory measurements eGFR (mL/min/1.73 m2), mean (SD)52.4 (17.5)85.3 (15.2)53.3 (4.1)38.8 (4.1)21.5 (7.2) Albuminuria Normal15,490 (11.0)–11,452 (15.1)3410 (12.2)628 (6.8) Moderately increased30,563 (21.7)15,338 (83.9)4030 (5.3)2136 (7.6)800 (8.7) Severely increased7717 (5.4)2937 (16.0)1151 (1.5)985 (3.5)904 (9.8) Not measured87,010 (61.8)–58,772 (77.9)21,398 (76.6)6840 (74.5)Comorbid conditions Atrial fibrillation18,844 (13.4)1589 (8.6)10,574 (14.0)4604 (16.4)1363 (14.8) Cardiovascular disease37,204 (26.4)3839 (21.0)19,534 (25.9)8769 (31.4)2967 (32.3) Congestive heart failure13,779 (9.7)850 (4.5)6716 (8.9)4183 (14.9)1558 (16.9) Diabetes45,714 (32.4)12,107 (66.2)17,341 (23.0)7019 (25.1)2605 (28.4) Hypertension91,181 (64.7)12,012 (65.7)48,309 (64.0)18,554 (66.4)5883 (64.1)Blood pressure (mmHg) mean (SD) Systolic135.9 (18.5)137.3 (17.7)135.2 (18.1)135.7 (19.5)136.1 (21.1) Diastolic74.9 (11.6)78.5 (11.5)74.5 (11.1)72.9 (11.7)72.7 (12.8)eGFR = estimated glomerular filtration rate, SD = Standard deviation.Missing data at baseline: Remoteness (684), IRSAD Quintile (658), eGFR (9999), smoking (9625), baseline blood pressure (17,053).a2016 Australian Statistical Geography Standard (ASGS) Remoteness Area (remoteness defined as major cities, regional [inner and outer] and remote [remote and very remote]).b2016 Index of Relative Socio-Economic Advantage and Disadvantage (IRSAD).\n",
      "\n",
      "\n",
      "**Domain 1: CKD monitoring**\n",
      "\n",
      "Overall, complete measurement of the core monitoring elements was achieved in 34.2% of the cohort (Fig. 1a). Measurement of the individual components was variable: 88.7% of patients had their blood pressure measured, 86.0% received an eGFR test, 41.1% received a UACR test, 70.9% had their lipid levels measured and 85.5% received an HbA1c test. Complete core monitoring was more frequent among patients with comorbidities, with highest completion rates in those with hypertension and diabetes (61.5%; Fig. 1b). Similarly, patients with diabetes more commonly had monitoring completed across the individual outcome measures of blood pressure, eGFR, UACR and lipids. This was most pronounced for albuminuria monitoring, with 69.6% of patients with diabetes having a recorded measure of UACR. Monitoring was higher in males compared to females (38.8% compared to 29.9%, p < 0.001, Supplementary Figure S3a).Fig. 1Percentage of patients achieving core monitoring (overall [A] and by comorbid hypertension and diabetes status [B]). CKD = chronic kidney disease, HTN = hypertension, DM = diabetes mellitus, ∗only reported in patients with diabetes mellitus; differences in percentage of patients achieving completion of core monitoring observed in Fig. 1b across all comorbidity subgroups were statistically significant (p < 0.0001).\n",
      "\n",
      "\n",
      "**Domain 2: cardiovascular risk management**\n",
      "\n",
      "ACE inhibitor or ARBs were prescribed at least once in 65.2% of patients during follow up and statin prescription in 54.4% of patients 50 years old and over or under 50 with additional cardiovascular risk factors (Fig. 2). Prescription of ACEi/ARB and statin was higher in those with comorbid diabetes and/or hypertension compared to those with CKD alone (highest in those with both hypertension and diabetes: 82.4% and 70.3%, respectively; Fig. 2). Prescription of ACEi/ARBs and statins were higher in males compared to females (Supplementary Figure S4a).Fig. 2Percentage of patients prescribed ACE inhibitor or ARB and a statin (overall and by comorbid hypertension and diabetes status). CKD = chronic kidney disease, HTN = hypertension, DM = diabetes mellitus, ∗statin in patients aged 50 and over or under 50 with additional cardiovascular risk factors; differences in medication prescription observed in across comorbidity subgroups were statistically significant (p < 0.0001).\n",
      "\n",
      "Amongst those who received a blood pressure measurement, 57.9% achieved a last recorded blood pressure of <140/90 mmHg, and 29.3% achieved a target of <130/80 mmHg (Fig. 3a). In patients with diabetes and/or moderately or severely increased albuminuria, 27.8% achieved a blood pressure <130/80 mmHg. In contrast to other aspects of monitoring, the achievement of blood pressure target <130/80 mmHg was highest in the group without diabetes or hypertension (36.5%), compared with hypertension (26.0%), diabetes (33.7%), and diabetes and hypertension (26.5%). Among patients with an LDL measurement, 38.8% achieved an LDL of <2 mmol/L (Fig. 3b). LDL <2 mmol/L target achievement was highest in patients with diabetes and hypertension (52.8%) and diabetes (48.1%), compared with patients with hypertension (31.1%) or CKD alone (27.9%).Fig. 3Percentage of patients achieving blood pressure target (A) of <140/90 mmHg and <130/80 mmHg, and LDL target of <2 mmol/L (B) (overall by comorbid hypertension and diabetes status). CKD = chronic kidney disease, HTN = hypertension, DM = diabetes mellitus; differences in percentage of patients achieving blood pressure target (2A) across all comorbidity subgroups were statistically significant (CKD + DM < 140/90, p = 0.0014; all other categories p < 0.0001); difference in percentage of patients achieving lipid target (2B) across all comorbidity subgroups were statistically significant (p < 0.0001).\n",
      "\n",
      "\n",
      "**Temporal trends in CKD care**\n",
      "\n",
      "Analysis by year of cohort entry showed an overall increasing trend in the proportion of patients satisfying the core monitoring criteria over the follow-up period (p for trend <0.0001; Fig. 4a). Improvements in monitoring were largely driven by a small increase in albuminuria monitoring over time (Fig. 4a). In contrast, ACE inhibitor or ARB use decreased over time from 71.7% in 2011 to 61.9% in 2020 (p for trend <0.0001, Fig. 4b). There were fluctuations in achievement of blood pressure target <140/90 mmHg and BP < 130/80 mmHg and LDL <2 mmol/L over time (Fig. 4b).Fig. 4Temporal trends in CKD monitoring (A) and cardiovascular risk management (B).\n",
      "\n",
      "\n",
      "**Factors associated with completion of core monitoring and medication prescription**\n",
      "\n",
      "Fig. 5 shows the relationships between patient characteristics and completion of core monitoring. Increasing age was associated with an increased relative risk of core monitoring completion (reference 18–29 years, RR [95% CI, p values]: 30–39 years 1.65 [1.35–2.02, <0.0001], 40–49 years 2.01 [1.65–2.43, <0.0001], 50–59 years 2.24 [1.85–2.70, <0.0001], 60–69 years 2.38 [1.97–2.87, <0.0001], 70–79 years 2.35 [1.95–2.84, <0.0001] and ≥80 years 1.72 [1.42–2.07, <0.0001]). Female sex was associated with a lower relative risk of core monitoring completion (RR 0.96 [95% CI: 0.95–0.98, p < 0.0001]). Compared with CKD stage 1–2, worsening eGFR stage was associated with a lower risk of core monitoring completion with CKD 3a and CKD 4–5 (RRs [95% CI, p value] for CKD stage 3A, 3B and 4–5: 0.91 [0.89–0.94, <0.0001], 1.01 [0.98–1.04, 0.2898], 0.84 [0.81–0.88, <0.0001], respectively). The presence of diabetes (RR 1.77 [95% CI 1.74–1.81, p < 0.0001]) and hypertension (RR 1.11 [95% CI 1.09–1.12, p < 0.0001]) was associated with increased relative risk of complete core monitoring. Overall, associations were similar across unadjusted and adjusted models (Supplementary Figure S5), and when individual components of the core monitoring outcome were assessed separately (Supplementary Table S1).Fig. 5Adjusted risk ratios (RR) for core monitoring among people with CKD. Modified Poisson regression adjusted for all characteristics assessing the relationship between sociodemographic and clinical characteristics and the risk ratios of core monitoring. eGFR = estimated glomerular filtration rate, UACR = urine albumin creatinine ratio, IRSAD = The Index of Relative Socio-economic Advantage and Disadvantage, RR = risk ratio, CI = confidence interval. p values: female sex (<0.0001), age (all categories, <0.0001), rurality (regional, 0.0326; remote, 0.3140), IRSAD (all categories, <0.0001), smoking status (ex-smoker, 0.0070; smoker, <0.0001), eGFR category (45–59, <0.0001; 30–44, 0.2898; <30 mL/min/1.73 m2, <0.0001), UACR category (moderately increased, 0.0005; severely increased, <0.0001; not measured, <0.0001), atrial fibrillation (0.2088), cardiovascular disease (0.1515), congestive heart failure (0.0001), diabetes (<0.0001), hypertension (<0.0001).\n",
      "\n",
      "Similar trends were seen in adjusted analysis of ACE inhibitor or ARB use and statin use, with baseline medication use having the strongest association with prescription during follow-up (Supplementary Table S2). Compared with normal UACR, the absence of albuminuria measurement was associated with a reduced relative risk of having ACE inhibitor or ARB prescription (RR 0.98 [95% CI 0.97–0.98, p < 0.0001]) and achieving blood pressure targets (BP < 140/90 mmHg RR 0.96 [95% CI 0.95–0.98, p < 0.0001] and BP < 130/80 mmHg RR 0.95 [95% CI 0.92–0.98, p = 0.0010]). Compared with normal UACR, severely increased UACR was associated with increased relative risk of having ACE inhibitor or ARB prescription (RR 1.03 [95% CI 1.01–1.05, p < 0.0001]). Achievement of blood pressure targets were negatively associated with increasing age, as was female sex and a history of hypertension (Supplementary Table S2).\n",
      "\n",
      "\n",
      "**Sensitivity analysis**\n",
      "\n",
      "In the 12 months following the index date, the completion of core monitoring was 24.5% (Supplementary Figure S6a and b). Overall results on the relationship between patient factors and completion of core monitoring remained largely unchanged (Supplementary Figure S7). Modified Poisson models with multiple imputation of missing values across core monitoring and cardiovascular risk monitoring also remained largely unchanged from the complete case analyses (Supplementary Table S3).\n",
      "\n",
      "\n",
      "**Discussion**\n",
      "\n",
      "In a large, nationally representative cohort of over 140,000 patients with CKD, we observed variation in CKD monitoring and cardiovascular risk management. Completion of blood pressure and renal function monitoring was high, while albuminuria monitoring and ACEi/ARB and statin prescription was relatively lower. Complete core monitoring was achieved in 34.2% of patients over a period of 18 months, which was largely attributable to low UACR testing with high completion of the other components. ACE inhibitors or ARBs were prescribed in 65.2% of patients, and statins in 54.4% of patients aged ≥50 years or <50 years with additional risk factors. Approximately two thirds of patients achieved a blood pressure target of <140/90 mmHg, while a target of <130/80 mmHg was achieved in under one third of patients. These findings, in combination with the increasing burden of CKD in Australia,5,21 highlight the need to implement effective strategies that improve the care and outcomes of patients with CKD.\n",
      "\n",
      "The extent of guideline-concordant CKD care observed in the current study varied across the care domains. We identified low rates of albuminuria monitoring (41.1%), consistent with accumulating reports indicating suboptimal albuminuria measurement.8,11,22,23 UACR testing is essential in enabling risk stratification and prognostication, and in guiding medication management.24 Indeed, when measured, UACR results were associated with increased use of appropriate disease modifying medications, with severely increased UACR being associated with higher rates of ACE inhibitor or ARB use compared to normal or moderately increased albuminuria.\n",
      "\n",
      "In considering strategies to improve UACR testing it is important to address previous barriers to testing in patients with CKD including lack of perceived management benefits and guideline recommendations,25 despite recommendations to the contrary.6,7 We defined CKD based on a biochemical diagnosis based on prior work on MedicineInsight finding significant under-coding in the medical record,26 in comparison to good sensitivity and specificity for diabetes coding.27 Improving clinician recognition and coding is a potential strategy to improve care with prior research finding a coded diagnosis of CKD in the medical record was associated with high rates of UACR testing28 and guideline directed care.28,29 Additionally, the use of clinical decision support software has been shown in some cases to improve UACR testing.30,31 Encouragingly, the rate of albuminuria monitoring and subsequently core monitoring was higher in those with a history of diabetes (69.6%), and there was a trend to increased core monitoring over time which was largely driven by a small increase in albuminuria monitoring. These findings suggest increasing uptake of local and international initiatives to improve diabetes care32, 33, 34 in clinical practice. Primary care practices were also eligible for an incentive primary care payment based on achieving the diabetes cycle of care (i.e., yearly albuminuria, eGFR, lipids and blood pressure checks).35 Effective strategies for achieving improvements across the broader population of patients with CKD are needed.\n",
      "\n",
      "On the other hand, we observed that blood pressure and eGFR monitoring was completed in a high portion of patients with CKD in primary care, 88.7% and 86.0% respectively. This is consistent with international cohorts with estimates ranging from 70 to 98% for blood pressure monitoring8,10,11,36,37 and 70–93% for renal function monitoring.8,10,11,36,38, 39, 40 However, high rates of blood pressure monitoring did not appear to translate to blood pressure target achievement, with 57.9% of the cohort achieving the target BP < 140/90 and 29.3% BP < 130/80 mmHg. We do note however, that office blood pressure is not the gold standard, and it is unknown what proportion of patients with hypertension underwent 24-h ambulatory blood pressure monitoring.\n",
      "\n",
      "Furthermore, whilst the use of ACE inhibitors and ARBs at 65.2% is greater than8,36,41,42 or similar to9,43,44 other international cohorts, temporal trends showed a small but steady decline in ACE inhibitor or ARB prescription over time. This has been previously described internationally.45,46 The reasons for this trend are not entirely clear, however barriers to ACE inhibitor or ARB initiation and ongoing prescription include previous acute kidney injury, hyperkalaemia, advancing CKD and multimorbidity.47\n",
      "\n",
      "There has been a significant expansion in the armamentaria of medications available to delay CKD progression,48, 49, 50 which are recommended in stepwise addition to ACE inhibitors and ARBs. Implementation of these medications with significant benefit in reducing morbidity and mortality associated with CKD is essential. A whole-systems approach is required to address the barriers and ensure implementation of CKD therapies, from public policy and health system reform through to provider and patient education.51,52\n",
      "\n",
      "Notably, we found a lower relative risk of receiving CKD monitoring and cardiovascular risk management in females. Previously, it was thought that sex differences in care may be attributable to co-morbidity burden and slower CKD progression.53 Our findings persisted despite adjusting for age, comorbidities and CKD stage. This has previously been reported with prior studies finding similar differences in CKD care that persist after adjusting for clinical and socio-demographic characteristics.8,54 The reasons for this are still to be established and this finding warrants further research to explore potential reasons for this.\n",
      "\n",
      "This study has several strengths including a contemporary cohort, the large community based nationally representative data source allowing for greater generalisability of results and longitudinal analysis. However, there are several limitations. The data reflect primary care data, and thus is likely to be incomplete particularly in relation to those with more severe disease. Patients who are attending specialist care, recommended in those with CKD stage 4 or 5 or severely increased albuminuria, are likely to be receiving monitoring at specialist services that may not be captured in primary care medical records and likely explains the association with decreased monitoring in more advanced CKD in this study. This is only a small subset of the patients however with only 7.0% having stage 4–5 CKD in this cohort. The MedicineInsight data came from an opt in program, and thus used a non-random recruitment strategy. Broadly, the data are consistent with the larger Australian population,13 however, sampling differences across Australian states and territories cannot be excluded. There is the potential for unmeasured confounding based both on uncaptured data and other confounders not analysed. Co-morbidity and observational data are extracted from coded or charted data within the medical record, and thus if entered as free text this would not be captured. There is also potential for data duplication if patients are attending multiple participating practices, although, given the general practice setting, significant movement across sites is not expected. It is possible that for some patients, CKD definition was first met prior to the study period. Our study may thus include patterns of CKD management at the time of both incident and prevalent CKD. Furthermore, the final years of follow-up occurred during the COVID-19 pandemic with reduced access to non-emergency health services, which likely explains the slight decrease in monitoring following CKD identification in 2019 and 2020.\n",
      "\n",
      "\n",
      "**Conclusion**\n",
      "\n",
      "This study highlights the priority areas for CKD care in a large nationally representative cohort. Whilst there were high rates of monitoring of blood pressure, eGFR and HbA1c, there is need to improve albuminuria monitoring which is essential for risk stratification and required for current CKD medication eligibility. Additionally, increasing uptake of ACE inhibitors and ARBs is a priority as they have been the standard for CKD care and the base medication to which newer guideline directed care is added. Further work is required to understand translation of CKD recognition and monitoring, into comprehensive management, and more broadly implementation strategies to improve all facets of CKD care.\n",
      "\n",
      "\n",
      "**Contributors**\n",
      "\n",
      "HW, JW, SK, PR and MJ contributed to the concept and rationale for the study and interpretation of the results. HW, JW, PR, SK, and MJ developed the study protocol and oversaw the implementation of the study analytical plan. HW drafted the initial manuscript, and MJ revised the initial draft. HW and MJ had access to all of the raw study data, verified the data and take responsibility for the integrity of the data. All authors contributed to the design of the study, interpretation of the data and, critical revision of the manuscript and approved the final manuscript for submission.\n",
      "\n",
      "\n",
      "**Data sharing statement**\n",
      "\n",
      "The current study is based on data from MedicineInsight, a national general practice data source developed by NPS MedicineWise and managed by the Australian Commission on Safety and Quality in Health Care.14 All relevant data are within the manuscript and its supplemental material.\n",
      "\n",
      "\n",
      "**Declaration of interests**\n",
      "\n",
      "The Renal Division of The George Institute for Global Health has received sponsorship funding provided by Boehringer Ingelheim and Eli Lilly Alliance and is supported by the University of New South Wales Scientia Program. The design, analysis, interpretation or writing of this manuscript was performed independent of all funding bodies. All study authors assumed final responsibility for all aspects of the study, including the decision to submit the manuscript for publication. MJ is responsible for research projects that have received research funding from Boehringer Ingelheim and Eli Lilly Alliance. MJ received research advisory group fees from NPS MedicineWise for a project outside the current study paid to his institution. SK has received consulting fees from Amgen, Chinook/Novartis and Boehringer and Ingelheim. BLN has received fees for travel support, advisory boards, publication support, scientific presentations and steering committee roles from AstraZeneca, Alexion, Bayer, Boehringer and Ingelheim, CSL-Behring, CSL-Seqirus, HCP Live, Medscape, Menarini, Novo Nordisk, Travere Therapeutics and Cornerstone Medical Education. SVB has received research grants from the National Health and Medical Research Council of Australia, advisory board fees from Bayer, AstraZeneca, and Vifor Pharma; speaker's honoraria from Bayer, AstraZeneca, Vifor Pharma, and Pfizer; and nonfinancial research support from Bayer, all fees paid to The George Institute for Global Health. CN has received project grants from Safer Care Victoria. DP has received research grants from the National Health and Medical Research Council of Australia.\n",
      "---------------------------------\n"
     ]
    }
   ],
   "source": [
    "pipeline(\"pfizer\", 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d9b6341b",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "research-paper-copilot-xnqBW2pO-py3.12",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
